<!DOCTYPE html>
<html>
  <head>
    <title>IWBDA: International Workshop on Biodesign Automation</title>
    <link href="http://fonts.googleapis.com/css?family=Exo|Open+Sans:400,600" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/bootstrap.min.css" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/main.css" media="screen" rel="stylesheet" type="text/css" />
    <script src="../javascripts/jquery-1.9.0.min.js" type="text/javascript"></script>
    <script src="../javascripts/bootstrap.min.js" type="text/javascript"></script>
  </head>
  <body>
    <div id="strip"></div>
    <div class="container">
      <div class="row" id="logo">
        <div class="span8">
          <a class="brand" href="/2015/"><h3>IWBDA 2015</h3></a>
        </div>
        <div class="span4">
          <h4 class="pull-right">August 19-21, 2015</h4>
        </div>
      </div>
      <div class="row" id="nav">
        <div class="span12">
          <ul class="nav nav-pills">
            <li><a href="/2015/#content">About</a></li>
            <li><a href="/2015/#reg">Registration</a></li>
            <li><a href="/2015/#hotel">Accommodations</a></li>
            <li><a href="/2015/#sponsors">Sponsors</a></li>
            <li><a href="/2015/#keynotes">Keynote Speakers</a></li>
	    <li><a href="/2015/#cfp">Call for Papers</a></li>
            <li><a href="/2015/sponsorship/">Sponsorship</a></li>
            <li class="active"><a href="/2015/program">Program</a></li>
            <li><a href="/2015/#scholarships">Scholarships</a></li>
            <li><a href="/2015/#special">Special Issue</a></li>
            <li><a href="/2015/#organizing">Contact</a></li>
            <li><a href="/2015/#related">Related Groups</a></li>
            <li><a href="/archive/">Archive</a></li>
          </ul>
        </div>
      </div>
      <div class="row">
	<div class="jumbotron">
          <div class="span6">
            <img src="../images/iwbda2015_teal.png" alt="IWBDA 2015: 7th International Workshop on Bio-Design Automation"/>
          </div>
	  <!--	<div class="span6">
		<h2>IWBDA 2015</h2>
		<h2>Seattle, WA</h2>
	  </div>-->
          <div class="span6" style="padding-top:75px">
            <h2>7th International Workshop</h2>
            <h2>on Bio-Design Automation</h2>
          </div>
	</div>
      </div>

      <br>

      <div id="content">

	<h4 id="proceedings">Proceedings</h4>
	<p><a href="/2015/program/IWBDA_2015_Proceedings.pdf">Link to proceedings pdf.</a></p>
	<hr>

	<h4 id="toc_0">Program</h4>

	<p>
	  The following is the preliminary schedule, subject to
	  change.<br>  
	  The schedule of abstracts to be presented
	  as <a href="#talks">talks</a>
	  and <a href="#posters">posters</a>. Detailed <a href="#instructions">instructions
	  for presenters</a> are below.
	</p>

	<hr>
	<h4 id="talks">Agenda</h4>(Updated on August 10, 2015)
	
	<p>
	  <table cellpadding=0 cellspacing=0 border=0 width=100%>

	    <tr>
              <th class=session colspan=2 align=left>Wednesday, August 19th</th>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>10:00 - 11:00</td>
	      <td>BDAC Executive Board Meeting (closed)</td>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>11:00 - 12:00</td>
	      <td>BDAC Membership Meeting (open)</td>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>12:00 - 13:00</td>
	      <td>Lunch break (continue BDAC discussion as needed) (open)</td>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>13:00 - 13:30</td>
	      <td>IWBDA/OWBDA 2016 Discussion (open)</td>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>13:30 - 17:30</td>
	      <td>SBOL Community Outreach Meeting (Coffee Break in the middle) (open)</td>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>13:30 - 14:00</td>
	      <td>&nbsp;&nbsp;&nbsp;Introduction to SBOL</td>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>14:00 - 15:00</td>
	      <td>&nbsp;&nbsp;&nbsp;Live demos of SBOL-based tools</td>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>15:00 - 15:30</td>
	      <td>&nbsp;&nbsp;&nbsp;Coffee Break</td>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>15:30 - 17:00</td>
	      <td>&nbsp;&nbsp;&nbsp;Input from the Bench</td>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>17:00 - 17:30</td>
	      <td>&nbsp;&nbsp;&nbsp;Roadmap and integration with other communities and standards</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th class=session colspan=2 align=left>Thursday, August 20th</th>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>8:00 - 8:30</td>
	      <td>Breakfast and Registration</td>
	    </tr>
	    <tr>
              <td width=12% valign=top align=center>8:30 - 8:40</td>
	      <td>Opening Remarks<br/>
		Douglas Densmore, General Chair</td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr>
              <th class=session colspan=2>Talk Session I: Standards & Data Exchange, Moderator: Aaron Adler</th>
	    </tr>
	    <tr>
              <td valign=top align=center>8:40 - 9:00</td>
	      <td>Progress from the Synthetic Biology Standards Consortium<br/>
		Matthew Munson, Sarah Munro and Marc Salit
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:00 - 9:20</td>
	      <td>The Synthetic Biology Open Language 2.0<br/>
		Bryan Bartley, Jacob Beal, Kevin Clancy, Goksel Misirli, Nicholas Roehner, Matthew Pocock, Tramy Nguyen, Zhen Zhang, Chris Myers, John Gennari, Herbert Sauro, Curtis Madsen, Anil Wipat and Ernst Oberortner
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:20 - 9:40</td>
	      <td>A Converter from the Systems Biology Markup Language to the Synthetic Biology Open Language<br/>
		Tramy Nguyen and Chris Myers
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:40 - 10:00</td>
	      <td>SBOL Stack: The One-stop-shop for Storing and Publishing Synthetic Biology Designs<br/>
		Curtis Madsen, Goksel Misirli, Matthew Pocock, Jennifer Hallinan and Anil Wipat
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>10:00 - 10:30</td>
	      <td>Coffee Break</td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr><th class=session colspan=2>Keynote</th></tr>
	    <tr>
              <td valign=top align=center>10:30 - 11:30</td>
	      <td><span class="speaker">Eric Klavins, University of
	      Washington</span> Programmable Synthetic Biology.<br>
	      Abstract: The design-build-test cycle in synthetic
	      biology requires build and test pipelines that are
	      highly reproducible, maximally informative, and
	      scalable. Traditional laboratory practices where a a
	      small team of researchers designs and performs their own
	      experiments in a unstructured lab, in contrast, are
	      difficult to reproduce and scale. In this talk, I will
	      describe recent efforts to standardize how protocols and
	      workflows are specified using formal, executable
	      programming languages. I will argue that treating
	      experimental workflows as code is leading to a
	      revolution in how experimental work is done and offers
	      huge advantages, such as easy integration into upstream
	      design software and easy collaboration. I will focus in
	      particular on my lab's experience with Aquarium, our
	      human-in-the-loop laboratory operating system, that can
	      be used to run experimental workflow programs reliably
	      and reproducibly. I will describe several projects in
	      yeast synthetic biology that Aquarium has enabled,
	      including cell-cell communication with auxin, large
	      CRISPR based gene circuits, and epigenetic memory
	      devices.</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Discussion Session I, Leader: Evan Appleton</th></tr>
	    <tr>
              <td valign=top align=center>11:30 - 12:30</td>
	      <td>Topic: Cloud Labs vs. Desktop Robots: Can there be only one?</td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Lunch</th></tr>
	    <tr>
              <td valign=top align=center>12:30 - 13:00</td>
	      <td>Lunch</td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr><th class=session colspan=2>Poster Session and Demos</th></tr>
	    <tr>
              <td valign=top align=center>13:00 - 14:30</td>
	      <td>Poster Session</td>
	    </tr>
            
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr>
              <th class=session colspan=2>Talk Session II: Pathway and Oligo Design, Moderator: Natasa Miskov-Zivanov</th>
	    </tr>
	    <tr>
              <td valign=top align=center>14:30 - 14:50</td>
	      <td>Stoichiometrically Minimal Source Pathways via Model Checking<br/>
		Matthew Fong and Sanjit Seshia
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>14:50 - 15:10</td>
	      <td>Double Dutch: A Tool for Designing Libraries of Variant Metabolic Pathways<br/>
		Nicholas Roehner and Douglas Densmore
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>15:10 - 15:30</td>
	      <td>Millstone: Software for iterative genome engineering<br/>
		Gleb Kuznetsov, Daniel B. Goodman, Marc J. Lajoie, George M. Church, Kevin Y. Chen, Changping Chen, Michael G. Napolitano and Brian W. Ahern
	      </td>
	    </tr>
	    
	    <tr>
              <td valign=top align=center>15:30 - 16:00</td>
	      <td>Coffee Break</td>
	    </tr>

	    <tr><th class=session colspan=2>Lab Tour</th></tr>
	    <tr>
              <td valign=top align=center>16:00 - 17:00</td>
	      <td>Klavins lab tour</td>
	    </tr>
            
	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th class=session colspan=2>Evening Activites</th>
	    </tr>
	    <tr>
              <td valign=top align=center>17:00 - 21:00</td>
	      <td>Dinner at UW Club</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th colspan=2 align=left>Friday, August 21st</th>
	    </tr>

	    <tr>
	      <td width=12% valign=top align=center>8:00 - 8:30</td><td>Breakfast and Registration</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th class=session colspan=2>Talk Session III: Process Management, Moderator: Nic Roehner</th>
	    </tr>
	    <tr>
              <td valign=top align=center>8:30 - 8:50</td>
	      <td>MERLIN: a DNA Design Tool for Large-Scale Genome Engineering<br/>
		Michael Quintin, Aaron Lewis, Natalie Ma, Douglas Densmore and Farren Isaacs
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>8:50 - 9:10</td>
	      <td>Software for Engineering Biology in a Multi-Purpose Foundry<br/>
		Benjie Chen, Dan Cahoon, Barry Canton and Austin Che
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:10 - 9:30</td>
	      <td>YeastFabCAM: Computer Assisted Manufacturing for constructing large-scale genetic parts registries<br/>
		Yisha Luo, Yue Shen, Emily Scher, Junbiao Dai and Yizhi Cai
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:30 - 9:50</td>
	      <td>Successful Failure: Best Practices for Quality Control of Large-scale DNA Assembly<br/>
		Bryan Bartley, Michal Galdzicki, John Gennari and Herbert Sauro
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:50 - 10:20</td>
	      <td>Coffee Break</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th class=session colspan=2>Industry Showcase, Moderator: Jake Beal</th>
	    </tr>
	    <tr>
              <td valign=top align=center>10:20 - 10:40</td>
	      <td>Realizing the dream of clean, structured data: Bringing manufacturing-grade quality to R&amp;D<br/>
		Timothy Gardner, Riffyn
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>10:40 - 11:00</td>
	      <td>TBD<br/>
		Chris Grant, Synthace
	      </td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr><th class=session colspan=2>Keynote</th></tr>
	    <tr>
              <td valign=top align=center>11:00 - 12:00</td>
	      <td><span class="speaker">Miriah Meyer, University of
	      Utah</span> Why an (interactive) visualization is worth
	      a thousand numbers.<br>  Astract: The advancement of techniques
	      and methods for collecting data has fundamentally
	      changed how we study life and all of its
	      complexities. But generating data is only the first step
	      --- developing methods to make sense of vast collections
	      of information is now widely considered the major
	      challenge. A key component of addressing this challenge
	      is visualization, which supports sense-making by
	      representing data as pictures and supporting exploration
	      through human-computer interactions. In this talk I'll
	      discuss how we design interactive visualizations and how
	      scientists use these tools to glean insight from complex
	      data.
              </td>

	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Lunch</th></tr>
	    <tr>
              <td valign=top align=center>12:00 - 13:30</td>
	      <td>Lunch</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Discussion Session II, Leader: Doug Densmore</th></tr>
	    <tr>
              <td valign=top align=center>13:30 - 14:30</td>
	      <td>Topic: BDA Commercialization Business Models</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th class=session colspan=2>Talk Session IV: Genetic Circuits I, Moderator: Traci Haddock</th>
	    </tr>
	    <tr>
              <td valign=top align=center>14:30 - 14:50</td>
	      <td>Automated design of genetic logic circuits<br/>
		Bryan Der, Douglas Densmore and Christopher Voigt
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>14:50 - 15:10</td>
	      <td>Parameter inference for gene circuit models<br/>
		Linh Huynh and Ilias Tagkopoulos
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>15:10 - 15:30</td>
	      <td>Design of Biological Circuits Using Signal-to-Noise Ratio<br/>
		Jacob Beal
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>15:30 - 16:00</td>
	      <td>Coffee Break</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr>
              <th class=session colspan=2>Talk Session V: Genetic Circuits II, Moderator: Kevin LeShane</th>
	    </tr>
	    <tr>
              <td valign=top align=center>16:00 - 16:20</td>
	      <td>SynBad: An SVP Design Framework<br/>
		Owen Gilfellon, Goksel Misirli, Curtis Madsen, Jennifer Hallinan, Paolo Zuliani and Anil Wipat
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>16:20 - 16:40</td>
	      <td>D-VASim: Dynamic Virtual Analyzer and Simulator for Genetic Circuits<br/>
		Hasan Baig and Jan Madsen
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>16:40 - 17:00</td>
	      <td>Fluigi: An Automated Framework for Creating Bioelectronic Devices<br/>
		Haiyao Huang, Aaron Heuckroth, Ryan J. Silva and Douglas Densmore
	      </td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Closing remarks</th></tr>
	    <tr>
              <td valign=top align=center>17:00 - 17:15</td>
	      <td>Closing remarks<br/>
		Douglas Densmore</td>
	    </tr>
	  </table>

	  <hr/>

	<h4 id="posters">Posters</h4>
	<p>
          <table cellpadding=10 cellspacing=0 border=0 width=100%>                                                                
            <tr><td>* Adaptive imaging cytometry to estimate parameters of gene networks models in systems and synthetic biology<br/>
		David Ball, Matthew Lux, Neil Adames, and Jean Peccoud</td></tr>
	    
	    <tr><td>archiYEAST: a command-line synthetic yeast architect<br/>
		Laura Adam and Eric Klavins</td></tr>

	    <tr><td>Big Mechanism Design and Analysis Automation<br/>
		Anuva Kulkarni, Cheryl Telmer, and Natasa Miskov-Zivanov</td></tr>

	    <tr><td>On the complexity of codon context optimization<br/>
		Dimitris Papamichail, Hongmei Liu, and Georgios Papamichail</td></tr>

	    <tr><td>Context Aware Pipetting<br/>
		Charles Fracchia, Joseph Jacobson, and George Church</td></tr>
	    
	    <tr><td>CRISPR and TAL Search and Design Tools for Cell Engineering<br/>
		Daniel Williams, Sridhar Ranganathan, and Joel Brockman</td></tr>
	    
	    <tr><td>Design and Characterization of Genetic Circuits using Multiplex DNA Synthesis<br/>
		Daniel Goodman, Casper Enghuus, and George Church</td></tr>
	    
	    <tr><td>* Design framework to program prokaryotic cell with diversified digital operations<br/>
		Adison Wong, Chueh Loo Poh, and Richard Kitney</td></tr>
	    
	    <tr><td>Design optimizations of precise synthetic genome targeting and editing small molecules for diverse disease loci<br/>
		Faisal Reza and Peter M. Glazer</td></tr>
	    
	    <tr><td>A Detailed, flexible standard for sharing DNA concepts<br/>
		Barbara Frewen, Jed Dean, and Aaron Kimball</td></tr>
	    
	    <tr><td>Efficient Analysis of SBML Models Using Arrays<br/>
		Leandro Watanabe and Chris Myers</td></tr>
	    
	    <tr><td>Extending the features and improving the performance of gro simulator: new bacterial conjugation and gene expression modules<br/>
		Mart&iacute;n Guti&eacute;rrez, Paula Gregorio-Godoy, Guillermo P&eacute;rez Del Pulgar, and Alfonso Rodr&iacute;guez-Pat&oacute;n</td></tr>
	    
	    <tr><td>* GenoLIB: a database of biological parts derived from a library of common plasmid features<br/>
		Neil R. Adames, Mandy L. Wilson, Gang Fang, Matthew W Lux, Benjamin Glick, and Jean Peccoud</td></tr>
	    
	    <tr><td>Phagebook: A Software Environment for Social Synthetic Biology<br/>
		Kathleen Lewis, Inna Turshudzhyan, Kara Le Fort, Nicholas Musella, Nicholas Roehner, Prashant Vaidyanathan, and Douglas Densmore</td></tr>
	    
	    <tr><td>Phoenix: An automated design-build-test tool<br/>
		Evan Appleton, Yash Agarwal, Zachary Chapasko, Ernst Oberortner, Alan Pacheco, Prashant Vaidyanathan, Nicholas Roehner, and Douglas Densmore</td></tr>
	    
	    <tr><td>Pooled, in situ assembly of complex genomic libraries using sorter-assisted genome engineering<br/>
		Robert Egbert, Eric Yu, and Adam Arkin</td></tr>
	    
	    <tr><td>Scylax&trade; - Automated design of cell factories incorporating synthetic enzymes<br/>
		Michal Galdzicki, Kyle Medley, Rudesh D Toofanny, Stanley Gu, Yih-En Andrew Ban, Herbert M Sauro, and Alexandre Zanghellini</td></tr>
	    
	    <tr><td>Towards Semi-Automated Experimental Design Using Model Inference in Synthetic Biology<br/>
		Tileli Amimeur, Eric Klavins</td></tr>
	    
	    <tr><td>Towards a sequence-level DNA design specification language<br/>
		Nicholas Bolten and Eric Klavins</td></tr>

	    <tr><td>* = Previously published work, not included in proceedings</td></tr>

          </table>

	  <hr/>

	  <h4 id="sponsor_posters">Sponsor Posters</h4>
	  
	<p/>

        <table cellpadding=10 cellspacing=0 border=0 width=100%>                                                                
          <tr><td>Autodesk</td></tr>
          <tr><td>SynBERC</td></tr>
	</table>

        <hr/>



	  <h4 id="instructions">Instructions for Oral and Poster Presentations</h4>
	  <h5>Oral presentations</h5>
	<p>Oral presentations will each be 15 minutes + 5 minutes of Q&A. 
	  <!-- We will be providing a projector and a laptop computer
	       for the presentations. Please bring your slides on a USB stick
	       at least 10 minutes before your session begins so that we can
	       load them on our laptop computer to minimize delay between
	       presentations. -->
	</p>
	
	<h5>Poster presentations</h5>
	<p><b>Posters should be in a horizontal orientation, 40 inches wide by 30 inches tall.</b></p>
	<hr/>
	
      </div>
    </div>
  </body>
</html>
