<!DOCTYPE html>
<html><head>
        <title>IWBDA: International Workshop on Biodesign Automation</title>
        <link href="http://fonts.googleapis.com/css?family=Exo|Open+Sans:400,600" media="screen" rel="stylesheet" type="text/css" />
        <link href="/2013/stylesheets/bootstrap.min.css" media="screen" rel="stylesheet" type="text/css" />
        <link href="/2013/stylesheets/main.css" media="screen" rel="stylesheet" type="text/css" />
        <script src="/2013/javascripts/jquery-1.9.0.min.js" type="text/javascript"></script>
        <script src="/2013/javascripts/bootstrap.min.js" type="text/javascript"></script>
    </head>
    <body>
        <div id="strip"></div>
        <div class="container">
            <div class="row" id="logo">
                <div class="span8"><a class="brand" href="/"><h3>IWBDA 2013</h3></a></div>
                <div class="span4"><h4 class="pull-right">July 12-13, 2013</h4></div>
            </div>
            <div class="row" id="nav"><div class="span12">
                    <ul class="nav nav-pills">
                        <li><a href="/2013/">About</a></li>
                        <li><a href="/2013/accommodations/">Accommodations</a></li>
                        <li><a href="/2013/#toc_11">Scholarships</a></li>
                        <li><a href="/2013/#toc_2">Registration</a></li>
                        <li class="active"><a href="/2013/program/">Program</a></li>
                        <li><a href="/2013/proceedings/">Proceedings</a></li>
                        <li><a href="/2013/acssynbio/">ACS Synthetic Biology Special Issue</a></li>
			<li><a href="/archive/">Archive</a></li>
                    </ul>
                </div>
            </div>
            <div class="row">
                <div class="jumbotron">
                    <div class="span6"><img src="/2013/images/iwbdalogo.jpg" /></div>
                    <div class="span6" style="padding-top:75px">
                        <h2>5th International Workshop</h2>
                        <h2>on Bio-Design Automation</h2>
                    </div>
                </div>
            </div>
            
            <div id="content">
                <h4 id="toc_0">Program</h4>
		<p>The schedule of abstracts to be presented as <a href="#toc_1">talks</a> and <a href="#posters">posters</a>. Detailed <a href="#instructions">instructions for presenters</a> are below.</p>
		<h5 id="toc_2">Location</h5>
		<p>
		Clore Lecture Hall, ground floor of the Huxley Building, Imperial College, London, UK.<br/>

		The poster session will be in Huxley 341/342, with poster set up open from 8:30am Friday morning.<br/>
		<br/>
		All tea and lunch breaks will take place in Huxley 341.<br/>
		Directions: <a herf="../Directions.pdf" download="newfilename">Map</a> 
		
		</p>
                <hr>
                <h4 id="toc_1">Agenda</h4>(Updated on July 13, 2013)

                <p>
                <table cellpadding=0 cellspacing=0 border=0 width=100%>
                    <tr>
                        <th class=session colspan=2 align=left>Friday, July 12th</th>
                    </tr>
		    <tr>
			    <td width=12% valign=top align=center>8:00 - 9:00</td><td>Registration</td>
		    </tr>
                    <tr>
                        <td width=12% valign=top align=center>9:00 - 9:15</td><td>Opening Remarks</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr><th class=session colspan=2>Keynote</th></tr>
                    <tr>
                        <td valign=top align=center>9:15 - 10:15</td><td><span class="speaker">Domitilla Del Vecchio</span>. From Context-dependence to Modularity in Biological Circuits</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>10:15 - 10:30</td><td>Coffee Break</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <th class=session colspan=2>Talk Session 1: Engineering, Parts, and Standardization</th>
                    </tr>
                    <tr>
                        <td valign=top align=center>10:30 - 10:55</td><td> (17) Michal Galdzicki, Ernst Oberortner, Mathew Pocock, Jacqueline Quinn, Mandy Wilson, Evan Appleton, Bryan Bartley, Jacob Beal, Swapnil Bhatia, Robert Cox, Raik Grunberg, Goskel Misirli, Hector Plahar, Nicholas Roehner, Larisa Soldotova, Guy-Bart Stan, Doug Densmore, <span class="speaker">Chris Myers</span>, Herbert Sauro and Anil Wipat. <br/>Recent Advances in the Synthetic Biology Open Language</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
			    <td valign=top align=center>10:55 - 11:20</td><td> (30) <span class="speaker">Jacqueline Quinn</span>, Jacob Beal, Swapnil Bhatia, Patrick Cai, Joanna Chen, Kevin Clancy, Robert Sidney Cox, Michal Galdzicki, Nathan Hillson, Akshay Maheshwari, Chris Myers, Umesh P, Matthew Pocock, Cesar Rodriguez, Herbert Sauro, Larisa Soldatova, Guy-Bart Stan, Mandy Wilson, Drew Endy.<br/> Synthetic Biology Open Language Visual: An Open-Source Graphical Notation for Synthetic Biology</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>11:20 - 11:45</td><td> (28) <span class="speaker">Michal Galdzicki</span>, Bryan A. Bartley, Sean C. Sleight, Evren Sirin and John Gennari.<br/> Version Control for Synthetic Biology</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
		    <td valign=top align=center>11:45 - 12:10</td><td> (33) <span class="speaker">David Kong</span>, Jonathan Babb and Todd Thorsen, Scott Wick, Jeremy Gam, Ron Weiss and Peter Carr.<br/> Microfluidic Genetic Circuit Assembly</td>

                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>12:10 - 13:30</td><td>Lunch</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr><th class=session colspan=2>Keynote</th></tr>
                    <tr>
                        <td valign=top align=center>13:30 - 14:30</td><td><span class="speaker">Alfonso Jaramillo</span>. Computational design of RNA circuits for signal processing in bacteria</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <th class=session colspan=2>Talk Session 2: Characterization and System Identification</th>
                    </tr>
                    <tr>
                        <td valign=top align=center>14:30 - 14:55</td><td> (23)<span class="speaker"> Claudio Angione</span>, Jole Costanza, Giovanni Carapezza, Pietro Lio' and Giuseppe Nicosia.<br/> Identifiability Analysis to Characterize Mitochondrial Diseases</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
		    <td valign=top align=center>14:55 - 15:20</td><td> (24) <span class="speaker">Boyan Yordanov</span>, Neil Dalchau, Paul Grant, Jim Haseloff, Stephen Emmott and Andrew Phillips.<br/> Automated Ratiometric Characterization using GEC</td>

                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>15:20 - 15:35</td><td> Coffee Break</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr><th class=session colspan=2>Keynote</th></tr>
                    <tr>
                    <td valign=top align=center>15:35 - 16:35</td><td><span class="speaker">Richard Kitney</span>. Foundational Resources in Synthetic Biology.</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr><th class=session colspan=2>Industry Panel</th></tr>
		    <tr><td valign=top align=center>16:35 - 17:35</td>
		    <td>Panel topic: The Entrepreneurial Spirit in BioDesign Automation: from startups to spinouts to internal innovation.</br>
				    Jacob Beal - BBN</br>
				    Jaqueline Quinn - Autodesk</br>
				    Wolf-Ekkehard Matzke - MINRES Technologies</br>
				    Debra Conway - Cybio</br>
				    Malcom Wilson - Cronus Technologies on behalf of Hudson Robotics</br>
				    Jim Hollenhorst - Agilent</br></td>
                    </tr>

                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>18:30 - 20:00</td><td>Dinner</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <th colspan=2 align=left>Saturday, July 13th</th>
                    </tr>
		    <tr>
			    <td width=12% valign=top align=center>8:30 - 9:00</td><td>Registration</td>
		    </tr>
                    <tr><th class=session colspan=2>Keynote</th></tr>
                    <tr>
                        <td valign=top align=center>9:00 - 10:00</td><td><span class="speaker">Jeff Hasty</span>. Engineered Gene Circuits: From Oscillators to Synchronized Clocks and Biopixels</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>10:00 - 10:15</td><td> Coffee Break</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <th class=session colspan=2>Talk Session 3: CAD Tools for Synthetic Biology</th>
                    </tr>
                    <tr>
                        <td valign=top align=center>10:15 - 10:40</td><td> (19) <span class="speaker">Evan Appleton</span>, Jenhan Tao and Douglas Densmore.<br/> Interactive Experimentally Driven Algorithms for Optimized DNA Assembly</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>10:40 - 11:05</td><td> (16) Linh Huynh and <span class="speaker">Ilias Tagkopoulos</span>.<br/> Optimizing module matching for synthetic gene circuit design automation</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>11:05 - 11:30</td><td> (21) <span class="speaker">Nicholas Roehner</span> and Chris Myers.<br/> Directed Acyclic Graph-Based Technology Mapping of Genetic Circuit Models</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>11:30 - 11:55</td><td> (13) Jonathan Blakes, <span class="speaker">Ofir Raz</span>, Natalio Krasnogor and Ehud Shapiro.<br/> A New Algorithm for Combinatorial DNA Library Assembly</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
			    <td valign=top align=center>11:55 - 13:30</td><td> Lunch and <a href="#posters">Poster session</a></td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <th class=session colspan=2>Talk Session 4: Biological Circuit Design and Assembly</th>
                    </tr>
                    <tr>
		    <td valign=top align=center>13:30 - 13:55</td><td> (26) Noah Davidsohn, <span class="speaker">Jacob Beal</span>, Aaron Adler, Fusun Yaman, Yinqing Li, Zhen Xie and Ron Weiss.<br/> Accurate Predictions of Genetic Circuit Behavior from Part Characterization and Modular Composition</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>13:55 - 14:20</td><td> (12) Maximilian Weitz, Jongmin Kim, Korbinian Kapsner, Erik Winfree, <span class="speaker">Elisa Franco</span> and Friedrich Simmel.<br/> Dynamical diversity of a programmable molecular oscillator in droplet microreactors</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
			    <td valign=top align=center>14:20 - 14:45</td><td> (27) <span class="speaker">Michael Guinn</span>, Yi Li and Leonidas Bleris.<br/> A 2-input Decoder Circuit in Human Cells</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>14:45 - 15:00</td><td> Coffee Break</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>

                    <tr><th class=session colspan=2>Keynote</th></tr>
                    <tr>
                        <td valign=top align=center>15:00 - 16:00</td><td><span class="speaker">Howard Salis</span>. Clone Less, Know More: Efficient Optimization of Circuits and Pathways with Next Generation Biophysical Modeling</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <th class=session colspan=2>Talk Session 5: Applications</th>
                    </tr>
                    <tr>
                        <td valign=top align=center>16:00 - 16:25</td><td> (14) <span class="speaker">Wolf-Ekkehard Matzke</span>, Michael Stumpf and Thorsten Mascher.<br/> Towards Industrial Strength BDA - A Short Essay</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>16:25 - 16:50</td><td> (4) <span class="speaker">Pablo Carbonell</span>, Pierre Parutto and Jean-Loup Faulon.<br/> RetroPath: Automated Pipeline for Embedded Metabolic Circuits.</td>
                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr>
                        <td valign=top align=center>16:50 - 17:15 </td><td>(32) <span class="speaker">Jole Costanza</span>, Claudio Angione, Giovanni Carapezza, Pietro Lio' and Giuseppe Nicosia.<br/> Multi-objective Optimization of Escherichia coli for Direct Production of 1,4-butanediol</td>

                    </tr>
                    <tr><td>&nbsp;</td><td></td></tr>
                    <tr><th class=session colspan=2>Closing remarks</th></tr>
                    <tr>
                        <td valign=top align=center>17:15 - 17:30</td><td>&nbsp;</td>
                    </tr>
                </table>
                <hr/>

		<h4 id="posters">Posters</h4>

                <p>
                <table cellpadding=10 cellspacing=0 border=0 width=100%>                                                                
                    <tr><td>(9) Joanna Chen, Rafael Rosengarten, Douglas Densmore, Timothy Ham, Jay Keasling and <span class="speaker">Nathan Hillson</span>. <br/>&emsp;j5 and DeviceEditor: DNA assembly design automation</td></tr>
                    <tr><td>(25) <span class="speaker">Jole Costanza</span>, Claudio Angione, Giovanni Carapezza, Pietro Lio' and Giuseppe Nicosia. <br/>&emsp;Pareto-based Gene and Pathway Sieve for Biofuel Production</td></tr>
                    <tr><td>(20) <span class="speaker">Vishwesh Kulkarni</span>, Evgeny Kharisov, Naira Hovakimyan, Marc Riedel and Jongmin Kim.<br/>&emsp;Load Capacity Improvements in DSD Systems Using Partially Open Feedback Control</td></tr>
                    <tr><td>(31) Ruei-Yang Huang, De-An Huang, Hui-Ju Katherine Chiang, <span class="speaker">Jie-Hong Roland Jiang</span> and Francois Fages.<br/>&emsp; Species Minimization in Computation with Biochemical Reactions</td></tr>
                    <tr><td>(22) <span class="speaker">Eric Kelsic</span>, Hattie Chung, Harris Wang and Roy Kishony.<br/>&emsp; Position-specific codon bias within an essential gene of E. coli</td></tr>
                    <tr><td>(1) <span class="speaker">Tuval Yehezkel</span>.<br/>&emsp; Development and application of automated DNA editing technology</td></tr>
                    <tr><td>(6) <span class="speaker">Hector Plahar</span>, Timothy Ham, Joanna Chen and Nathan Hillson.<br/>&emsp; JBEI-ICE: An Open Source Biological Part Registry</td></tr>
                    <tr><td>(15) <span class="speaker">Kevin Clancy</span>, Kok Hien Gan, Brandon Ang, Kin Chong Sam, Yong Ming Chen, Reina Angelica, Bee Im Teo, Wolfgang Strenzl, Michael Muth, Brynne Stanton, Virgil Rhodius, Ying-Ja Chen, David Raab, Lian Seng Loh, Chris Voigt, Douglas Densmore and Todd Peterson.<br/>&emsp; Vector NTI® Express Designer - a new bioinformatics platform for synthetic biology design</td></tr>
                    <tr><td>(34) <span class="speaker">Arturo Casini</span>, Tom Ellis and Geoff Baldwin. Computationally designed DNA linkers to enhance a novel automated DNA assembly standard</td></tr>
		    <tr><td>(2) <span class="speaker">Gregory Linshiz</span>, <span class="speaker">Nina Stawski</span>, Sean Poust, Changhao Bi, Jay Keasling and Nathan Hillson.<br/>&emsp; PR-PR: Cross-platform standard language for automation of Synthetic Biology research</td></tr>
		    <tr><td>(7) Sayane Shome, Pablo Cárdenas, <span class="speaker">Pedro Fernandes</span>, Gerd Moe-Behrens and Alioune Ngom.<br/>&emsp; SynBio App Selector: An Application Selection Engine for Synthetic Biology</td></tr>
		    <tr><td>(8) Pablo Cárdenas, Sayane Shome, <span class="speaker">Pedro Fernandes</span>, Alioune Ngom and Gerd Moe-Behrens.<br/>&emsp; SynBrick: A Crowdsourcing Game for Synthetic Biology</td></tr>
		    <tr><td>(29) <span class="speaker">Rishi Ganguly</span>, Peng Liu, Swapnil Bhatia, Christopher Voigt and Douglas Densmore.<br/>&emsp; Cello: A software workflow for modular design and compilation of genetic circuits</td></tr>
		    <tr><td>(18) <span class="speaker">Aaron Adler</span>, Fusun Yaman and Jacob Beal. <br/>&emsp;Online Tools for Characterization, Design, and Debugging.</td></tr>
		    <tr><td>(5) Eduardo Abeluik, Yuri Benda&ntilde;a, Rodrigo Pavez, Nathan Hillson and <span class="speaker">Michael Fero</span>.<br/>&emsp; A NoSQL approach to a synthetic biology database.</td></tr>
		    <tr><td>(10) <span class="speaker">Vishwesh Kulkarni</span>, Boyan Yordanov, Andrew Phillips and Jongmin Kim.<br/>&emsp; Biomolecular Implementation of Linear Dynamical Systems.</td></tr>
		    <tr><td>(3) <span class="speaker"> Ehud Magal</span>.<br/>&emsp; Robot QC methods.</td></tr>
                </table>
                <hr/>
		<h4 id="instructions">Instructions for Oral and Poster Presentations</h4>
		<h5>Oral presentations</h5>
		<p>Oral presentations will each be 20 minutes + 5 minutes of Q&A. We will be providing a projector and a laptop computer for the presentations. Please bring your slides on a USB stick at least 10 minutes before your session begins so that we can load them on our laptop computer to minimize delay between presentations.</p>
		
		<h5>Poster presentations</h5>
		<p>The maximum poster size that can be displayed is:
		<ul>
			<li>A0: 1189 x 841 mm</li> or
			<li>46.8 x 33.1 in</li>
			<li>Orientation: Portrait</li>
		</ul>
		<hr/>
            </div>
        </div>
    </body>
</html>
