<!DOCTYPE html>
<html>
  <head>
    <title>IWBDA: International Workshop on Biodesign Automation</title>
    <link href="http://fonts.googleapis.com/css?family=Exo|Open+Sans:400,600" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/bootstrap.min.css" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/main.css" media="screen" rel="stylesheet" type="text/css" />
<!--    <script src="javascripts/jquery-3.1.0.min.js" type="text/javascript"></script>
    <script src="javascripts/bootstrap.min.js" type="text/javascript"></script> -->
  </head>
  <body>
    <div id="strip"></div>
    <div class="container">
      <div class="row" id="logo">
        <div class="span8">
          <a class="brand" href="http://www.iwbdaconf.org/2019/"><h3>IWBDA 2019</h3></a>
        </div>
        <div class="span4">
          <h4 class="pull-right">July 8 - July 10, 2019</h4>
        </div>
      </div>
      <div class="row" id="nav">
        <div class="span12">
          <ul class="nav nav-pills">
            <li class="active"><a href="/2019/#content">About</a></li>
            <li><a href="#cfp">Call for Papers</a></li> 
            <li><a href="#scholarships">Scholarships</a></li>
            <li><a href="#workshops">Workshops</a></li>                       
            <li><a href="#reg">Registration</a></li> 
            <li><a href="#acc">Accommodations</a></li>
            <li><a href="#organizing">Contact</a></li>
            <li><a href="#acssynbio">Special Issue</a></li>
            <li><a href="#related">Related Conferences</a></li>
            <li><a href="#archive">Archive</a></li>
          </ul>
        </div>
    </div>
    <div class="row">
      <div class="jumbotron">
	<div class="span12">
          <img src="../images/IWBDA2019.png" alt="IWBDA 2019: 11th International Workshop on Bio-Design Automation, Cambridge, UK"/>
        </div>
	<div class="span12">
	  <h3 class="pull-right">11th International Workshop on Bio-Design Automation</h3>
        </div>
      </div>
    </div>

    <h3>Vote on best poster and presentation <a href="https://forms.gle/463fg3ByAxAVxARx8">here</a></h3>

    <h4 id="proceedings">Proceedings</h4>
    <p><a href="../docs/IWBDA19Proceedings.pdf">IWBDA 2019 Booklet</a></p>
    <hr>
  
  <br>

<br>


<p><strong>Monday, July 8th</strong></p>

<p>09:00 - 17:30 <strong>SBOL Workshop</strong> (<u><a href="../docs/SBOLWorkshopProgram.pdf" rel="nofollow" target="_blank">Schedule</a></u>)</p>
<p>13:30 - 17:30 <strong>Workshop on Bio-Design for Portability (BD4P)</strong> (<u><a href="https://pzuliani.github.io/bd4p.html" rel="nofollow" target="_blank">Schedule</a></u>)</p>

<p><strong>Tuesday, July 9th</strong></p>

<p>08:00 - 09:00 <strong>Breakfast and Registration</strong></p>
<p>09:00 - 09:10 <strong>Welcome & Opening Remarks</strong> Pietro Lio' (University of Cambridge)</p>

<p>09:10 - 10:30 <strong>Session I: Design Automation</strong>, Chair: Bradley Brown (Newcastle University)</p>
<ul>
	<li><i>DNAWeaver: optimal DNA assembly strategies via supply networks and shortest-path algorithms</i><br/>
    Valentin Zulkower, The Edinburgh Genome Foundry Team, and Susan Rosser</li>
	<li><i>An automated QC pipeline for libraries with high degree of variants</i><br/>
    Ernst Oberortner, Robert Evans, and Jan-Fang Cheng</li>
	<li><i>Automated Design and Characterization of Large Toolboxes of Highly Non-Repetitive Genetic Parts</i></br>
    Ayaan Hossain and Howard M. Salis</li>
	<li><i>The Operon Refactoring and Construction Assistant (ORCA): Streamlined gene cluster refactoring</i><br/>
    Ernst Oberortner, Nathan J. Hillson, and Jan-Fang Cheng</li>
</ul>

<p>10:30 - 11:00 <strong>Break</strong></p>

<p>11:00 - 12:00 <strong>Keynote I: Luca Cardelli (University of Oxford)</strong></p>
<div style="text-align: justify">
<ul style="list-style-type:none;">
	<li><strong>Title:</strong>&nbsp;Integrated Scientific Modeling and Lab Automation</li>
	<li><strong>Abstract:</strong>&nbsp;The cycle of observation, hypothesis formulation, experimentation, and falsification that has driven scientific and technical progress is lately becoming automated in all its separate components. However, integration between these automated components is lacking. Theories are not placed in the same formal context as the(coded) protocols that are supposed to test them: neither description knows about the other, although they both try to describe the same process. 
    We develop integrated descriptions from which we can extract both the model of a phenomenon (for possibly automated mathematical analysis), and the step carried out to test it (for automated execution by lab equipment). This is essential if we want to carry out automated model synthesis, falsification, and inference, by taking into account uncertainties in both the model structure and in the equipment tolerances that may jointly affect the results of experiments.</li>
</ul>
</div>

<p>12:00 - 13:30 <strong>Lunch</strong></p>

<p>13:30 - 14:30 <strong>Session II: Machine Learning</strong>, Chair: Jacob Beal (Raytheon BBN Technologies)</p>
<ul>
	<li><i>Towards a framework for implementing evolutionary algorithms in bacterial colonies</i><br/>
    Javier Carri&oacute;n, Yerko Ortiz, and Mart&iacute;n Guti&eacute;rrez</li>
	<li><i>Combining metabolic modelling with machine learning accurately predicts yeast growth rate</i><br/>
    Christopher Culley, Supreeta Vijayakumar, Guido Zampieri, and Claudio Angione</li>
	<li><i>Towards Detection of Engineering in Metagenomic Sequencing Data for Yeast and Other Fungi</i></br>
    Sancar Adali, Aaron Adler, Joel S. Bader, John Grothendieck, Thomas Mitchell, Anton Persikov,
Jonathan Prokos, Richard Schwartz, Mona Singh, Allison Taggart, Benjamin Toll, Stavros Tsakalidis,
Daniel Wyschogrod, Fusun Yaman, Eric Young, and Nicholas Roehner</li>
</ul>

<p>14:30 - 15:00 <strong>Break</strong></p>

<p>15:00 - 15:40 <strong>Session III: Computation in Genetic Circuits</strong>, Chair: Prashant Vaidyanathan (Microsoft)</p>
<ul>
	<li><i>Biophysical Analysis for Implementing Neuro-inspired Computing in Living Cells</i><br/>
    Ramez Danial, Luna Rizik, Ximing Li, and Raghd Abu Sinni</li>
	<li><i>Analyzing Genetic Circuits for Hazards and Glitches</i><br/>
    Pedro Fontanarrosa, Hamid Hosseini, Amin Espah Borujeni, Yuval Dorfan, Chris Voigt, and Chris Myers</li>
</ul>

<p>15:40 - 16:10 <strong>Poster Pitches I (1min per poster)</strong>, Chair: Prashant Vaidyanathan (Microsoft)</p>
<ul>
	<li><i>A Logic Programming Language for Computational Nucleic Acid Devices</i><br/>
		Carlo Spaccassassi, Matthew R. Lakin, and Andrew Phillips
	</li>
  <li><i>CellScanner: a software for extracting physical features from individual cells</i><br/>
		Sebasti&aacute;n Ant&oacute;n, Marco Clavero, and Mart&iacute;n Guti&eacute;rrez
  </li>
  <li><i>OptBioDes: optimal design for the synbio toolchain</i><br/>
		Pablo Carbonell, Rainer Breitling, Jean-Loup Faulon, and The SYNBIOCHEM Team
  </li>
  <li><i>Integration of Performance Metrics into Microfluidic Design Automatiom</i><br/>
		Radhakrishna Sanka and Douglas Densmore
	</li>
  <li><i>Modular Microfluidic Design Automation Using Machine Learning</i><br/>
		Ali Lashkaripour, Christopher Rodriguez, Noushin Mehdipour, David McIntyre, and Douglas Densmore
  </li>
  <li><i>Accelerating the Threshold and Timing Analysis of Genetic Logic Circuit Models</i><br/>
		Sanaullah, Hasan Baig, and Jeong A Lee
  </li>
  <li><i>Better research by efficient sharing: evaluation of free management platforms for synthetic biology designs</i><br/>
	 	Uriel Urquiza-Garc&iacute;a, Tomasz Zieli'nski, and Andrew J. Millar
	</li>
  <li><i>A method for compiling arbitrary transfer functions to molecular circuits</i><br/>
		Iuliia Zarubiieva, Francesca Mantellino, Andrew Phillips, and Vishwesh Kulkarni
  </li>
  <li><i>The Morphogen Circuit Builder &amp; Compiler</i><br/>
	  	Bryan Bartley, Brian Basnight, Jesse Tordoff, Jacob Beal, Ron Weiss
  </li>
  <li><i>Machine Learning Algorithms for Robust Meta-Analysis of Gene Expressions</i><br/>
		Vishwesh Kulkarni, Xinwu Yu, and Weikang Qian
	</li>
  <li><i>Mutation of synthetic constructs in E. coli</i><br/>
		Duncan Ingram, Mark Isalan, and Guy-Bart Stan
  </li>
  <li><i>Parallel Binary Sorting and Shifting with DNA</i><br/>
		Tonglin Chen and Marc Riedel
	</li>
</ul>

<p>16:10 - 17:30 <strong>Poster Session </strong></p>


<p><strong>Wednesday, July 10th</strong></p>


<p>09:00 - 09:10 <strong>Welcome & Opening Remarks</strong> Prashant Vaidyanathan (Microsoft)</p>

<p>09:10 - 10:30 <strong>Session IV: Modelling</strong>, Chair: Nicholas Roehner (Raytheon BBN Technologies)</p>
<ul>
	<li><i>An automated model reduction tool to guide the design and analysis of synthetic biological circuits</i><br/>
		Ayush Pandey and Richard M. Murray</li>
	<li><i>Estimating Biologically Relevant Network Structures from Time-series Data</i><br/>
		Zoltan A. Tuza and Guy-Bart Stan</li>
	<li><i>Model-driven design of genetic regulatory networks using virtual parts</i></br>
		G&ouml;ksel Misirli, Bill Yang, and Anil Wipat</li>
	<li><i>Stochastic Analysis of an Genetic Sensor</i><br/>
		Jeanet Mante, Pedro Fontanarrosa, and Chris Myers</li>
</ul>

<p>10:30 - 11:00 <strong>Break</strong></p>

<p>11:00 - 12:00 <strong>Keynote II: Traci Haddock-Angelli (iGEM Foundation)</strong></p>
<div style="text-align: justify">
<ul style="list-style-type:none;">
  <li><strong>Title:</strong>&nbsp;Setting Standards in Synthetic Biology</li>
  <li><strong>Abstract:</strong>&nbsp;Since 2003, the iGEM Foundation has been working on setting standards in synthetic biology. 
From the early days of BioBricks, we have worked to introduce the idea of engineering standards to young synthetic biologists through the competition and to the rest of the synthetic biology community through the Registry of Standard Biological Parts. Recent work has focused on introducing a new assembly standard to the Registry and on the development of standard protocols to promote the use of absolute units for fluorescence measurements. Through the development of these standards, we know that community involvement is crucial to the creation and adoption of new standards, and invite you all to join us as we work towards creating a strong, responsible, and visionary synthetic biology industry.</li>
</ul>
</div>

<p>12:00 - 13:30 <strong>Lunch</strong></p>

<p>13:30 - 14:30 <strong>Session V: Data standards and Visualization</strong>, Chair: Ernst Oberortner (Berkeley Lab)</p>
<ul>
	<li><i>Visualization of Part Use in SynBioHub</i><br/>
		Jeanet Mante, Zach Zundel, and Chris Myers</li>
	<li><i>SBOL Visual 2 Ontology</i><br/>
		G&ouml;ksel Misirli, Jacob Beal, Thomas E. Gorochowski, Guy-Bart Stan, Anil Wipat, and Chris Myers</li>
	<li><i>Flapjack: an open-source tool for storing, visualising, analysing and modelling kinetic gene expression data</i><br/>
		Guillermo Y&aacute;&ntilde;ez, Isaac N&uacute;&ntilde;nez, Tamara Matute, Fern&aacute;n Federici, and Timothy J. Rudge</li>
</ul>

<p>14:30 - 15:00 <strong>Break</strong></p>

<p>15:00 - 16:00 <strong>Session VI: Microfluidics</strong>, Chair: Radhakrishna Sanka (Boston University)</p>
<ul>
	<li><i>A Reconfigurable Digital Microfluidics Platform</i><br/>
		Georgi Tanev, Luca Pezzarossa, Winnie E. Svendsen, and Jan Madsen</li>
	<li><i>Design Automation of Microfluidic Droplet Sorting Platforms</i><br/>
		David McIntyre and Douglas Densmore</li>
	<li><i>Detecting Engineering in Single Cells using Tapestri Microfluidics</i><br/>
		Aaron Adler, Adam Abate, Joe Collins, Ben Demaree, Kevin Keating, Xiangpeng Li, Thomas Mitchell, David Ruff, Allison Taggart, Shu Wang, Daniel Weisgerber, Daniel Wyschogrod, Fusun Yaman, Eric M. Young, and Nicholas Roehner</li>
</ul>


<p>16:00 - 16:30 <strong>Break</strong></p>

<p>16:30 - 17:10 <strong>Discussion/Special Session: Jacob Beal</strong></br>
  Topic: <i>“Keeping the Promises of Synthetic Biology.”</i>
</p>

<p>17:10 - 17:30 <strong>Wrap up, Announcements, &amp; Award Ceremony</strong></p>

</div></div></body></html>
