<!DOCTYPE html>
<html>
  <head>
    <title>IWBDA: International Workshop on Biodesign Automation</title>
    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
    <link href="http://fonts.googleapis.com/css?family=Exo|Open+Sans:400,600" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/bootstrap.min.css" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/main.css" media="screen" rel="stylesheet" type="text/css" />
<!--    <script src="javascripts/jquery-3.1.0.min.js" type="text/javascript"></script>
    <script src="javascripts/bootstrap.min.js" type="text/javascript"></script> -->
  </head>
  <body>
    <div id="strip"></div>
    <div class="container">
      <div class="row" id="logo">
        <div class="span8">
          <a class="brand" href="/2021/"><h3>IWBDA 2021</h3></a>
        </div>
        <div class="span4">
          <h4 class="pull-right">September 20 - September 24, 2021</h4>
        </div>
      </div>
      <div class="row" id="nav">
        <div class="span12">
          <ul class="nav nav-pills">
            <li class="active"><a href="/2021/#content">About</a></li>
	          <li><a href="/2021/#reg">Registration</a></li>
            <!--<li><a href="#program">Program</a></li>-->
            <li><a href="/2021/#cfp">Call for Papers</a></li>
            <li><a href="/2021/#cfw">Call for Workshops</a></li>
            <li><a href="/2021/program">Program</a></li>
            <li><a href="/2021/workshops">Workshops</a></li>
            <li><a href="/2021/acssynbio">Special Issue</a></li>
            <li><a href="/2021/#twitter">#IWBDA2021</a></li>
            <li><a href="/2021/#organizing">Contact</a></li>
            <li><a href="/2021/#archive">Archive</a></li>
          </ul>
        </div>
    </div>
    <div class="row">
      <div class="jumbotron">
	      <div class="span12">
	        <img src="../images/IWBDA2021.png" alt="IWBDA 2021: 13th International Workshop on Bio-Design Automation, Online"/>
        </div>
	      <div class="span12">
	        <h3 class="pull-right">13th International Workshop on Bio-Design Automation</h3>
        </div>
      </div>
    </div>

    
    <div class="span12">

      <!--Workshop 1-->
      <hr id="sbol3">
      <div class="card">
        <div class="card-header">
          <b>Workshop 1</b> SBOL Version 3: Data Exchange throughout the Bioengineering Lifecycle
        </div>
        <div class="card-body">
          <h5 class="card-title">Date/Time: Monday, 20 September 2021. 6:30pm - 8:30pm BST <a href="https://calendar.google.com/calendar/u/0/r/eventedit/copy/MWk0YnM1NzFzY2ozNGRhYmlsa29xYW02YTEgdWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZw" target="_blank">Copy to my Calendar</a></h5>
          <p class="card-text">The Synthetic Biology Open Language (SBOL) allows knowledge about biological designs to be captured using a machine-tractable, ontology-backed representation built on top of Semantic Web technologies. In addition to representing genetic designs, SBOL Version 3 can now be used to represent knowledge across multiple scales and throughout the entire synthetic biology workflow, from the specification of a single molecule or DNA fragment to tracking experimental workflows with complex samples or multicellular systems containing multiple interacting genetic circuits. This tutorial will discuss both the fundamentals of SBOL3 and experiences from its use in a variety of projects.</p>
          <p class="card-text">
          This tutorial will provide the following to attendees:
          <ul>
            <li>an introduction to the SBOL3 data model</li>
            <li>examples of workflows built on SBOL3</li>
            <li>hands-on experience with SBOL3 libraries and related tools</li>
            <li>an understanding of how to build extensions using SBOL Factory</li>
          </ul></p>
          <h5 class="card-title">Presenters:</h5>
          <ul>
            <li>Jacob Beal</li>
            <li>Bryan Bartley</li>
            <li>Goksel Misirli</li>
            <li>Tom Mitchell</li>
          </ul>
        </div>
      </div>

      <!--Workshop 2-->
      <hr id="biocrnpyler">
      <div class="card">
        <div class="card-header">
          <b>Workshop 2</b> From Chemical Reaction Network Compilation to Bayesian Parameter Inference
        </div>
        <div class="card-body">
          <h5 class="card-title">Date/Time: Part I - Tuesday, 21 September 2021. 5:45pm - 7:45pm BST <a href="https://calendar.google.com/calendar/u/0/r/eventedit/copy/NmRxY25hM3VhNXBibW5uZDllczJodDNuMGMgdWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZw" target="_blank">Copy to my Calendar</a></h5>
          <h5 class="card-title">Date/Time: Part II - Tuesday, 21 September 2021. 7:45pm - 9:45pm BST <a href="https://calendar.google.com/calendar/u/0/r/eventedit/copy/NTltZm1sdjMwaXNnOXZjNGJscWVsZWphaWsgdWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZw" target="_blank">Copy to my Calendar</a></h5>
          <p class="card-text">In this two part workshop, we begin with an introduction and interactive demo of the BioCRNpyler software package which lets modelers compile detailed chemical reaction networks from simple specifications. A case study of integrase circuits will be used to highlight the flexibility of this package. The second part of the workshop introduces and demos a black-box parameter inference pipeline using the AutoReduce, Bioscrape, and Emcee packages using data from cell free integrase circuit experiments. Note that each section is ~2 hours and will be designed so participants can come to either demonstration individually or attend both.</p>
          <h5 class="card-title">Presenters:</h5>
          <ul>
            <li>William Poole</li>
            <li>Ayush Pandey</li>
            <li>Andrey Shur</li>
            <li>Zoltan Tuza</li>
          </ul>
          <a href="../docs/CRNToBayesParamInference.pdf" rel="nofollow" target="_blank" class="btn btn-primary">Workshop Poster</a>
        </div>
      </div>

      <!--Workshop 3-->
      <hr id="mlsynbio">
      <div class="card">
        <div class="card-header">
          <b>Workshop 3</b> Machine Learning Aided Advances in Synthetic Biology
        </div>
        <div class="card-body">
          <h5 class="card-title">Date/Time: Wednesday, 22 September 2021. 4:00pm - 5:30pm BST <a href="https://calendar.google.com/calendar/u/0/r/eventedit/copy/MGRvbGYyMzNvZjlpdjc5MWZka2Zmb3BtZDcgdWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZw" target="_blank">Copy to my Calendar</a></h5>
          <p class="card-text">Synthetic biology aims to understand, refine, control, re-engineer, and evolve nature. Data is a key element in this process. Rapid progress in synthetic biology is making increasingly large, increasingly complex, and novel datasets available to us. These exciting developments have direct parallels in machine learning which is facilitating methods and paradigms that have increasingly superior computational speed as well as increasingly superior ability to transform the data into useful information. What are the possibilities if synthetic biology and machine learning come together with great synergy? In this workshop, we present four talks that answer some specifics for this exciting open ended question.</p>
          <h5 class="card-title">Presenters:</h5>
          <ul>
            <li>Sabyasachi Ghosh</li>
            <li>Manoj Gopalkrishnan</li>
            <li>Maziya Ibrahim</li>
            <li>Karthik Raman</li>
            <li>Jean-Loup Faulon</li>
            <li>Jonathan Tellechea</li>
            <li>Pablo Carbonell</li>
          </ul>
          <a href="../docs/MLinSynBio.pdf" rel="nofollow" target="_blank" class="btn btn-primary">Workshop Poster</a>
        </div>
      </div>

      <!--Workshop 4-->
      <hr id="3duf">
      <div class="card">
        <div class="card-header">
          <b>Workshop 4</b> DIY Microfluidics CAD - Extending 3DuF for fun and publications
        </div>
        <div class="card-body">
          <h5 class="card-title">Date/Time: Wednesday, 22 September 2021. 7:15pm - 9:15pm BST <a href="https://calendar.google.com/calendar/u/0/r/eventedit/copy/MDVnbXNudDJpbjRkM2hmM3NkcDNmbDlvdW0gdWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZw" target="_blank">Copy to my Calendar</a></h5>
          <p class="card-text">3DuF is an open source interactive design tool for developing microfluidics. It uses a standardized component library that allows microfluidics designs to utilize their favorite components and rapidly design their devices and send it out for manufacturing. In this workshop we will show one can quickly add a new component onto 3DuF and utilize it in your own workflow. We will also discuss how you can add more features onto the tool for your own publications, get code reviews, etc. using the tool.</p>
          <h5 class="card-title">Presenters:</h5>
          <ul>
            <li>Radhakrishna Sanka</li>
          </ul>
        </div>
      </div>

      <!--Workshop 5-->
      <hr id="flapjack">
      <div class="card">
        <div class="card-header">
          <b>Workshop 5</b> Flapjack: Data Management and Analysis for Genetic Circuit Characterization
        </div>
        <div class="card-body">
          <h5 class="card-title">Date/Time: Thursday, 23 September 2021. 5:15pm - 9:15pm BST <a href="https://calendar.google.com/calendar/u/0/r/eventedit/copy/MzJ1ZGFrMXJ2ODcwM2FpaHEwaWhhcTQ1dTAgdWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZw" target="_blank">Copy to my Calendar</a></h5>
          <p class="card-text"> <u>Flapjack intro:</u></p>
          <p class="card-text">
            Flapjack is a data management system for synthetic biology data. Our tool enables researchers to store, visualize, analyze and share genetic circuit data, linking the test phase with the build and learn phases from the Design-Build-Test-Learn (DBTL) cycle, helping automation of Synthetic Biology workflows. The system is implemented as a full-stack web application, consisting of a back-end developed as a REST and WebSockets API and a front-end user interface for easy access. The webapp is deployed using docker containers in a microservices architecture in order to ease installation. We also provide a python package to access programmatically to the API, giving flexibility to users to develop and integrate with other software tools or standards in the synthetic biology environment such as SBOL, SynBioHub, among others.
          </p>
          <p class="card-text"><u>Frontend tutorial:</u></p>
          <p class="card-text"><i>Presenter: Carlos Vidal-Céspedes</i></p>
          <p class="card-text">In the frontend tutorial we will take a walk-through to the user interface, cover how to format data to be able to upload it to the platform, how to share and delete studies and visualize queries and relevant analyzes to the data. We will also format and download the resulting plots and data.</p>
            
          <p class="card-text"><u>Python package:</u></p>
          <p class="card-text"><i>Presenter: Gonzalo Vidal</i></p>
          <p class="card-text">This tutorial will have an interactive session employing Google Collab, where attendees will have hands-on experience coding to upload, visualize, analyze and download data using our Flapjack python package to access the API. We will also make and format plots using the obtained data.</p>
          <p class="card-text">First, we will query and visualize data using interactive plots. Next, we will obtain plots in publication-like format. Then, we will analyze data fitting and parameterizing a hill function to an induction curve (sigmoidal response curve). Also, we will explore the data using Pandas functions. Finally we will use SBOL and SynBioHub synthetic biology tools as means of exemplifying the integration with other SynBio software tools using Python.</p>
  
          <p class="card-text"><u>Local installation:</u></p>
          <p class="card-text"><i>Presenter: Guillermo Yañez</i></p>
          <p class="card-text">In this part we will get over the installation process of a Flapjack instance in your local machine using docker containerization, covering all the three major OS (macOS, Linux and Windows).</p>
          <p class="card-text">We will first introduce Docker, then we will deploy Flapjack and finally we will make the initial configuration in order to have your local instance up and running. It will be required that attendees have Docker and Anaconda installed.</p>
          <p class="card-text">Install Docker (MacOS/Windows): <a href="https://www.docker.com/products/docker-desktop">Docker Desktop for Mac and Windows | Docker</a></p>
          <p class="card-text">Install Docker(Linux): <a href="https://docs.docker.com/engine/install/">Install Docker Engine | Docker Documentation</a></p>
          <p class="card-text">Install Anaconda: <a href="https://www.anaconda.com/products/individual-d#Downloads">Anaconda | Individual Edition</a></p>
          
          <br>
          <p class="card-text"><b>Outcomes</b></p>
          At the end of the tutorial, attendees will be able to:
          <ul>
            <li>Access and use Flapjack through the web app's user interface.</li>
            <li>Share data and work collaboratively.</li>
            <li>Visualize and analyze data using the Flapjack engine.</li>
            <li>Use pyFlapjack to access the API to store, explore, visualize, analyze and download data.</li>
            <li>Access SynBioHub instances in order to obtain genetic parts information.</li>
            <li>Install a local instance of Flapjack.</li>
          </ul>

          <h5 class="card-title">Presenters:</h5>
          <ul>
            <li>Carlos Vidal-Céspedes</li>
            <li>Gonzalo Vidal</li>
            <li>Guillermo Yáñez</li>
          </ul>
          <a href="../docs/FlapjackWorkshopPoster.pdf" rel="nofollow" target="_blank" class="btn btn-primary">Workshop Poster</a>
        </div>
      </div>
      
      <!--Workshop 6-->
      <hr id="bmss">
      <div class="card">
        <div class="card-header">
          <b>Workshop 6</b> BMSS - An Automated BioModel Selection System for Gene Circuit Designs
        </div>
        <div class="card-body">
          <h5 class="card-title">Date/Time: Friday, 24 September 2021. 2:05pm - 2:50pm BST <a href="https://calendar.google.com/calendar/u/0/r/eventedit/copy/M25ncWxzZmRlNmN2dmY4dTlvYzI2Z2Y4dnEgdWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZw" target="_blank">Copy to my Calendar</a></h5>
          <p class="card-text">
            Modelling plays an important role in Synthetic Biology in our continuous search for improved performance of our engineered biological systems. Kinetic models can provide insights into the dynamic behaviours of the genetic circuits and guide experimental designs. However, it can be challenging yet tedious to identify the “right” model that best describes experimental data obtained for a particular genetic circuit while tracing the entire model development and analysis. In this workshop, we will demonstrate an open-source python software package, BioModel selection system (BMSS) [1,2,3], which functions as a hypothesis-testing tool developed to perform automated model selection and enable a more systematic approach to the model development and analysis. BMSS ranks models from a library of pre-built models stored in the database (UBase/MBase) using experimental data. BMSS is designed to be modular and highly extensible. We will demonstrate the various features in BMSS using case studies. </p>
            <p class="card-text">
              The demonstration will cover
              <ul>
                <li>how model selection can be performed using BMSS</li>
                <li>how models can be incorporated into UBase/MBase for future selection</li>
                <li>how models can be analysed using BMSS</li>
                <li>other additional features available in BMSS</li>
              </ul>
            </p>
              
            <p class="card-text">
              References:
              <ol>
                <li>Ngo, Russell Jie Kai, et al. "BMSS2: a unified database-driven modelling tool for systematic model selection and identifiability analysis." bioRxiv (2021).</li>
                <li>Yeoh, Jing Wui, et al. "An automated biomodel selection system (BMSS) for gene circuit designs." ACS synthetic biology 8.7 (2019): 1484-1497.</li>
                <li>https://github.com/EngBioNUS/BMSS2</li>
              </ol>
            </p>
            <h5 class="card-title">Presenters:</h5>
            <ul>
              <li>Russell Jie Kai Ngo</li>
              <li>Jing Wui Yeoh</li>
            </ul>
          <a href="../docs/BMSS_workshop.pdf" rel="nofollow" target="_blank" class="btn btn-primary">Workshop Poster</a>        
        </div>
      </div>

      <!--Workshop 7-->
      <hr id="sbolvisual">
      <div class="card">
        <div class="card-header">
          <b>Workshop 7</b> Visualizing biological designs using SBOL visual
        </div>
        <div class="card-body">
          <h5 class="card-title">Date/Time: Friday, 24 September 2021. 4:00pm - 6:00pm BST <a href="https://calendar.google.com/calendar/u/0/r/eventedit/copy/NHFxaWQ3N3NzZnFxaGk2NjZsYzAxZmNzbnUgdWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZw" target="_blank">Copy to my Calendar</a></h5>
          <p class="card-text">As the complexity of our biological designs grow there is an increasing need to be able to visualize them in a clear and accessible way to aid communication and facilitate further development. SBOL visual aims to provide a standardised set of glyphs and conventions for how these are connected to address this need and is closely aligned to the SBOL data standard, enabling direct connections to biological designs in a machine-readable form. In this short workshop, we will introduce SBOL visual, discussing the underlying motivation, development, and current state of the standard as well as providing some hands-on demonstrations of recently released tools and programming libraries in which SBOLv diagrams can be built. The end of the workshop will be an open discussion around future directions for the standard and the supporting tools that will be needed to support this.</p>
          <h5 class="card-title">Presenters:</h5>
          <ul>
            <li>Thomas Gorochowski</li>
            <li>James Scott-Brown</li>
          </ul>
        </div>
      </div>

      <!--Workshop 8-->
      <hr id="circuitdesign">
      <div class="card">
        <div class="card-header">
          <b>Workshop 8</b> Principles of genetic circuit design: programming living cells to perform novel functions
        </div>
        <div class="card-body">
          <h5 class="card-title">Date/Time: Friday, 24 September 2021. 6:45pm - 8:45pm BST <a href="https://calendar.google.com/calendar/u/0/r/eventedit/copy/NXIydWk2M3JxZWtoaGEwM2xiMDZqY29mMjMgdWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZw" target="_blank">Copy to my Calendar</a></h5>
          <p class="card-text"> The "Principles of genetic circuit design: programming living cells to perform novel functions" workshop will give an overview on the design process of synthetic genetic circuits as gene expression control systems, composition of genetic circuits from different classes of gene regulators, different computer-aided design and modelling tools used in the genetic circuit design workflow, common failure modes encountered in genetic circuit design and engineering strategies to overcome these failure modes and optimize the genetic circuit design.
          </p>
          <h5 class="card-title">Presenters:</h5>
          <ul>
            <li>Hatem Mohamed Gaber Abdelrahman</li>
          </ul>
          <a href="../docs/GeneticCircuitDesign.pdf" rel="nofollow" target="_blank" class="btn btn-primary">Workshop Poster</a>
        </div>
      </div>
      <br>

    </div>

    


</br> 




</div></div></body></html>
