<!DOCTYPE html>
<html>
  <head>
    <title>IWBDA: International Workshop on Biodesign Automation</title>
    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
    <link href="http://fonts.googleapis.com/css?family=Exo|Open+Sans:400,600" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/bootstrap.min.css" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/main.css" media="screen" rel="stylesheet" type="text/css" />
<!--    <script src="javascripts/jquery-3.1.0.min.js" type="text/javascript"></script>
    <script src="javascripts/bootstrap.min.js" type="text/javascript"></script> -->
  </head>
  <body>
    <div id="strip"></div>
    <div class="container">
      <div class="row" id="logo">
        <div class="span8">
          <a class="brand" href="/2022/"><h3>IWBDA 2022</h3></a>
        </div>
        <div class="span4">
          <h4 class="pull-right">24-26 October, 2022</h4>
        </div>
      </div>
      <div class="row" id="nav">
        <div class="span12">
          <ul class="nav nav-pills">
            <li class="active"><a href="../index.html#content">About</a></li>
	    <li><a href="../index.html#reg">Registration</a></li>
            <li><a href="../index.html#cfp">Call for Papers</a></li>
            <li><a href="../index.html#sponsors">Sponsors</a></li>
            <li><a href="../webinars/index.html">Webinars</a></li>
            <!--<li><a href="./index.html#cfw">Call for Workshops</a></li>-->
            <li><a href="./index.html">Program</a></li>
            <!--<li><a href="/2022/workshops">Workshops</a></li>-->
            <li><a href="../index.html#special">Special Issue</a></li>
            <!--<li><a href="#proceedings">Proceedings</a></li>-->
            <!-- <li><a href="#twitter">#IWBDA2022</a></li> -->
            <li><a href="../index.html#organizing">Contact</a></li>
            <li><a href="../index.html#archive">Archive</a></li>
          </ul>
        </div>
    </div>

<br>
<!--
    <div class="row">
      <div class="jumbotron">
	      <div class="span12">
	        <img src="../images/IWBDA2022.png" alt="IWBDA 2021: 13th International Workshop on Bio-Design Automation, Online"/>
        </div>
	      <div class="span12">
	        <h3 class="pull-right">14th International Workshop on Bio-Design Automation</h3>
        </div>
      </div>
    </div>
    <div class="span12">
      <hr id="calendar">
      <h3>IWBDA 2021 Calendar</h3>
      <p>Please click on any event to see additional details or to add any session to your calendar.</p>
      <iframe src="https://calendar.google.com/calendar/embed?height=600&wkst=2&bgcolor=%23ffffff&ctz=Europe%2FLondon&src=dWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&color=%23D50000&showNav=0&mode=AGENDA&showCalendars=0" style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe>
-->
      
      <hr id="proceedings">
      <h3>IWBDA 2022 Proceedings</h3>
      <p>Full conference proceedings with the program committee information, the full program, and accepted 2-page abstracts is available as <a href="../docs/proceedings.pdf">PDF</a>.</p>

      <hr id="detailedprogram">
      <h3>IWBDA 2022 Detailed Program</h3>
      <p>All times CEST (UTC+1) Paris local time</p>

  <p><strong><u>Monday, 24th October 2022 (at Learning Planet Institute)</u></strong></p>

  <p>09:30 - 10:30 <strong>Registration</strong></p>

  <p>Due to presenter's unforeseen circumstances, the NONA Workshop 1 has to be cancelled. We apologize for the inconvenience.</p>

  <p><del>09:35 - 10:30 </del> <del><strong>Workshop 1: NONA: Open-Source Software in Synthetic Biology and Bio-design Automation</strong></del></p>
  <p><del>George Johnson and Vishwesh Kulkarni.</del></p>
  <p><del>This demo will highlight 5 open-source tools including Primer3Plus, openVectorEditor, SeqViz, Puppeteer, and PyHamilton and how to utilize them in an lab experiment from initial primer design to a scripted execution of the experiment.</del></p>

  <p>10:30 - 10:35 <strong>Welcome & Opening Remarks</strong></p>

  <p>10:35 - 11:45 <strong>Workshop 2: SBOL Version 3: Data Exchange throughout the Bioengineering Lifecycle</strong></p>
  <p>Tom Mitchell, Jacob Beal and Bryan Bartley.</p>
  <p>The Synthetic Biology Open Language (SBOL) allows knowledge about biological designs to be captured using a machine-tractable, ontology-backed representation built on top of Semantic Web technologies. In addition to representing genetic designs, SBOL Version 3 can now be used to represent knowledge across multiple scales and throughout the entire synthetic biology workflow, from the specification of a single molecule or DNA fragment to tracking experimental
workflows with complex samples or multicellular systems containing multiple interacting genetic circuits. This tutorial will discuss both the fundamentals of SBOL3 and experiences from its use in a variety of projects. This tutorial will provide the following to attendees:
• an introduction to the SBOL3 data model
• examples of workflows built on SBOL3
• hands-on experience with SBOL3 libraries and related tools
• an understanding of how to build extensions using SBOL Factor</p>

  <p>11:45 - 12:00 Short Break</p>

  <p>12:00 - 13:00 <strong>Workshop 2 (continued)</strong></p>

  <br>
  <p> 13:00 - 14h30 Lunch on your own</p>
  <br>

  <p>14:30 - 15:30 <strong>Workshop 3: Automating Laboratory Protocols with the Laboratory Open Protocol (LabOP) language</strong></p>
  <p>Bryan Bartley, Jacob Beal and Dan Bryce.</p>

  <p>In this workshop, we will introduce the LabOP (Laboratory Open Protocol) language and provide an opportunity for hands-on learning. The session will focus on demonstrations from LabOP language stakeholders, highlighting use cases from iGEM, Friendzymes, and Opentrons, including sample protocol specification, and tools to create and convert protocols into different protocol languages. The hands-on exercises will give participants the opportunity to author protocolsusing the Python LabOP programming API and the GUI-based protocol editor, PAMLed.</p>

  <p>15:30 - 15:45 Short Break</p>

  <p>15:45 - 16:45 <strong>Workshop 3 (continued) </strong></p>

  <p>16:45 - 17:00 Short Break</p>

  <p>17:00 - 17:30 <strong>Lightning Talks 1</strong> (90 seconds presentation of each one of the posters)</p>

  <p>17:30 - 18:30 <strong>Posters</strong></p>

  <ul>

      <li><p><strong>Steps Towards Functional Synthetic Biology.</strong> Ibrahim Aldulijan, Jacob Beal, Sonja Billerbeck, Jeff Bouffard, Gaël Chambonnier, Nikolaos Delkis, Isaac Guerreiro, Martin Holub Martin Holub, Daisuke Kiga, Jacky Loo, Paul Ross, Vinoo Selvarajah, Noah Sprent, Gonzalo Vidal and Alejandro Vignoni</p></li>

      <li><p><strong>Adapting Malware Detection to DNA Screening.</strong> Dan Wyschogrod, Jeff Manthey, Tom Mitchell, Steven Murphy, Adam Clore and Jacob Beal</p></li>

      <li><p><strong>Artificial Metabolic Networks: enabling neural computations with metabolic networks.</strong> Léon Faure and Jean-Loup Faulon</p></li>

      <li><p><strong>Developing a scoring system to optimise the design of CRISPR Cas12 diagnostics.</strong> Akashaditya Das and Ana Pascual-Garrigos</p> </li>

      <li><p><strong>DBTL bioengineering cycle: developing a population oscillator.</strong> Andrés Arboleda-García, Iván Alarcon-Ruiz, Yadira Boada, Jesús Picó and Eloisa Jantus-Lewintre</p> </li> 

      <li><p><strong>Computer-aided enhancement of genetic design data.</strong> Matthew Crowther and Angel Goñi-Moreno</p> </li>

      <!-- <li><p><strong>SynPath – An Automated Biosynthetic Pathway Design and Analysis Tool.</strong> Carol Gao, Helena van Tol and Xi Wang<p> </li> -->

      <li><p><strong>The Context Matrix: A Framework for Context-Aware Synthetic Biology.</strong> Camillo Moschner, Charlie Wedd and Somenath Bakshi<p> </li>

      <li><p><strong>An Interactive Microfluidic Design and Control Workflow.</strong> Yangruirui Zhou and Douglas Densmore</p> </li>

      <li><p><strong>Experimental Data Converter.</strong> Sai Samineni, Gonzalo Vidal, Jeanet Mante, Guillermo Yañez-Feliú, Carlos Vidal-Céspedes, Chris Myers and Timothy J. Rudge </li>

      <li><p><strong>Expanding the metaheuristic framework: evolving cells with the bat algorithm.</strong> Víctor Reyes, Nicolás Hidalgo and Martín Gutiérrez</p> </li>

      <li><p><strong>PLATERO: A Plate Reader Calibration Protocol to work with different instrument gains and asses measurement uncertainty.</strong>Yadira Boada, Alba González-Cebrián, Joan Borràs-Ferrís, Jesús Picó, Alberto Ferrer and Alejandro Vignoni</p> </li>

      <li><p><strong>Spatially Solving the Graph Coloring Problem Using Intercell Communication.</strong> Daniela Moreno, Diego Araya and Martín Gutiérrez</p> </li>

      <li><p><strong>Active Learning-based Optimal Experimental Design for efficient biomanufacturing.</strong> Iván Blázquez Arenas, Pablo Carbonell and Irene Otero-Muras</p> </li>

      <li><p><strong>Probabilistic programming for synthetic gene networks.</strong> Lewis Grozinger and Angel Goñi-Moreno</p> </li>

      <!-- <li><p><strong>In-silico design for fold-change detection (FCD) synthetic circuits.</strong> Rongying Huang and Ramez Daniel</p> </li> -->

      <li><p><strong>Rule-based generation of synthetic genetic circuits.</strong> Daisuke Kiga, Kazuteru Miyazaki, Shoya Yasuda, Ritsuki Hamada, Sota Okuda, Ryoji Sekine, Naoki Kodama and Masayuki Yamamura</p> </li>

      <li><p><strong>Standardizing the Representation of Parts and Devices for Build Planning.</strong> Jacob Beal, Vinoo Selvarajah, Gael Chambonnier, Traci Haddock-Angelli, Alejandro Vignoni, Gonzalo Vidal and Nicholas Roehner</p></li>
      <li><p><strong>Dynamic Behavior Alters Influences and Sensitivities in Biological Networks.</strong> Gaoxiang Zhou and Natasa Miskov-Zivanov</p> </li>
      <li><p><strong>Exploring Advantages and Limitations of Discrete Modeling of Biological Network Motifs</strong> Difei Tang, Gaoxiang Zhou and Natasa Miskov-Zivanov</p> </li>
  </ul>
  <hr>

<p><strong><u>Tuesday, 25th October 2022 (at Learning Planet Insitute)</u></strong></p>

  <p>08:30 - 09:00 <strong>Registration</strong></p>

  <p>09:00 - 10:00 <strong>Workshop 4: Principles of genetic circuit design</strong></p>

  <p>Hatem Abdelrahman.</p>
  <p>Principles of genetic circuit design: programming living cells to perform novel functions
  The workshop will give an overview on the design process of synthetic genetic circuits as gene expression control systems from design specification to build optimization, modular composition of genetic circuits from different classes of gene regulators, different computer-aided design and modelling tools used in the genetic circuit design workflow including genetic design automation (GDA), common failure modes encountered in genetic circuit design and engineering strategies to overcome these failure modes and optimize the genetic circuit design.</p>

  <p>10:00 - 10:30 <strong>Demo: DySE: Dynamic System Explanation framework</strong></p>

  <p>Gaoxiang Zhou, Difei Tang and Natasa Miskov-Zivanov.</p> 

  <p>In this demo, we will demonstrate our framework, DySE (Dynamic System Explanation), and in particular several tools: model simulation, model extension, interaction classification, interaction filtering, model checking and sensitivity analysis.</p>

  <p>10:30 - 10:45 Short Break</p>

  <p>10:45 - 11:45 <strong>Workshop 5: Python Tools for Modeling of Biocircuits From High-Level Specification to Parameter Inference</strong></p>

  <p>Ayush Pandey and Zoltan Tuza</p>

  <p>In this workshop, we present an interactive demo of three open-source Python software tools — (1) BioCRNpyler: to build mathematical models of biological circuits at varying levels of detail from high-level specifications, (2) AutoReduce: to automatically obtain reduced mathematical models, and (3): Bioscrape: for sensitivity analysis and parameter inference. The workshop is designed to provide a hands-on tutorial to the participants on building models of biological circuits in a systematic manner and validating the models with experimental data.</p>

  <p>Computational notebook on <a href="http://tiny.cc/iwbda-biocrnpyler" target="_blank">BioCRNpyler</a> for CRN modeling</p>

  <p>11:45 - 12:00 Short Break</p>

  <p>12:00 - 13:00 <strong>Workshop 5: Learning Parameters from Experimental Data using Bayesian Inference and Model Reduction</strong></p>
    <p>The second part of the workshop is designed to be independent. The focus will be on using experimental data to validate mathematical models using AutoReduce and Bioscrape.</p>
  <p>Computational notebook on <a href="http://tiny.cc/iwbda-autoreduce" target="_blank">model reduction</a> with AutoReduce</p>
  <p>Computational notebook on <a href="http://tiny.cc/iwbda-bioscrape" target="_blank">parameter inference</a> with Biosrape</p>
  <br>
  <p> 13:00 - 14h30 Lunch on your own</p>
  <br>

  <p>14:30 - 15:30 <strong>Workshop 6: OneModel: an easy-to-use tool for modular building of biocircuits models (Part 1)</strong></p>
  <p>Fernando N. Santos-Navarro.</p>

  <p>Part 1: Learn the basics of OneModel<p>

  <p>In this part, we will describe the basics of OneModel and how its syntax is structured. We will learn to define parameters, species, reactions, and rules (assignment, rate, and algebraic rules). And then how to export the model defined in OneModel syntax to SBML. Next, we will learn how to modularize our models. We will show how to define classes, extend them, and define methods that facilitate the connection of different models. Finally, we will show how to import previously defined models from GitHub and how we can integrate them into our custom models.</p>

  <p>Computational notebook on <a href="http://bit.ly/iwbda-onemodel" target="_blank">OneModel</a></p>

  <p>15:30 - 15:45 Short Break</p>

  <p>15:45 - 16:45 <strong>Workshop 6: OneModel: an easy-to-use tool for modular building of biocircuits models (Part 2)</strong></p>

  <p>Fernando N. Santos-Navarro.</p>

  <p>Part 2: The second part of the workshop builds on the first part and consists of learning how to use our host-aware model for E. coli. This host-aware model is implemented using the OneModel syntax, and it is freely available to the public.</p>


  <p>16:45 - 17:00 Short Break</p>

  <p id="keynote">17:00 - 18:00 <strong>Keynote: Adding automation and reactiveness to your experiments: motivation, tools and applications.</strong></p>

  <p><a href="../index.html#keynote">Gregory Batt</a></p>

  <p>
  Small-scale, low-cost bioreactors are emerging as powerful tools for microbial systems and synthetic biology research. They allow tight control of cell culture parameters over long durations. These unique features enable researchers to perform sophisticated experiments and to achieve high reproducibility.
  <br>
However, existing setups are limited in their measurement capabilities. It is often essential to follow over time key characteristics of the cultured cell population, such as gene expression levels, cellular stress levels, and cell size and morphology. Researchers usually need to manually extract, process and measure culture samples to run them through sensitive and specialized instruments. Manual interventions strongly constrains the available temporal resolution and reactiveness capabilities.
  <br>
In this talk, I will present ReacSight, a generic and flexible strategy to enhance bioreactor arrays with automated measurements capabilities and reactive experiment control. It can also be used to enhance any computer-controlled plate-based measurement device with pipetting capabilities and automation. ReacSight leverages the affordable Opentrons pipetting robots. It is ideally suited to integrate open-source, open-hardware components but can also accommodate closed-source, GUI-only components. Applications include the control of an artificial differentiation system in yeast to create consortia with tuneable composition, and the characterization of protein secretion under various stress conditions to optimize production in yeast.
  <br>
In the same spirit, we have also developed MicroMator, a software tool to streamline the use of Micromanager and enable the realization of smart, reactive microscopy experiments. MicroMator also fosters throughput, reproducibility and reactivity.
  </p>

  <p>
  </p>

  <hr>

<p><strong><u>Wednesday, 26th October 2022 (at iGEM Paris Expo - Porte de Versailles)</u></strong></p>


  <p>09:30 - 09:45 <strong>Welcome & Opening Remarks</strong></p>

  <p>09:45 - 10:00 <strong>Sponsored Talk: Asimov - Intelligent design of living systems to enable next-generation biotechnologies</strong></p>

  <p>Traci Haddock</p>

  <p>Abstract: Asimov is developing a new way to program living cells by building the first computer-aided design (CAD) platform for biology. Our goal is to build a true CAD platform for biology to accelerate advanced biotechnologies that can tackle some of the world’s biggest challenges. Our platform enables other biotech companies and labs to level up their ability to design and manufacture next-generation therapeutics.</p>

  <p>10:00 - 10:30 <strong>Lightning talks</strong> (90 seconds presentation of each one of the posters)</p>

  <p>10:30 - 10:45 Short Break</p>

  <p>10:45 - 11:45 <strong>Talks Session 1: Biofoundries and Automation</strong> Chair: Olivier Borkowski</p>

  <p>10:45 - 11:00 <strong>Efficient Droplet Microfluidic Characterization for Design Automation </strong></p>
  <p>Diana Arguijo and Douglas Densmore<p>
  <p>Abstract: Droplet microfluidics provides a tool for the acceleration of synthetic biology research by increasing screening throughput and reproducibility. Here, we propose a workflow for microfluidic device design automation that combines active machine learning, camera-less droplet monitoring, and automatic device characterization. Specifically, through the use of impedance sensing droplets in a microfluidic device can be monitored automatically.</p>

  <p>11:00 - 11:15 <strong>Low-cost Open Source Benchtop Bioreactor</strong>
  <p>Vitor Marchesan, Lívia Galinari, Luiza Possa, Tiago Mendes, João Vitor Molino and Livia Ferreira-Camargo</p>
  <p>Abstract: Automation and real-time data collection of controlled biological cultures can be a hassle for small businesses and research institutions due to costs of a benchtop bioreactors. This work proposes a multipurpose bioreactor that is accessible to those with tight budget, by using common labware associated with 3D printer parts, common house utilities, and runs Open-Source hardware and software. The reactor is designed to also work as a photobioreactor, can directly measure pH and temperature, estimate growth stage based on the medium color by turbidimetry, as well as operate as a batch or fed-batch bioreactor. Parameters such as heating, light cycle and pH can be programmed to desired values. The bioreactor control and programming are done using a Web browser on any computer on the same network.</p> 

  <p>11:15-11:30 <strong>Harnessing Biofoundries for the forward engineering of strains, with a focus on increased cis, cismuconic acid titers in yeast</strong></p>
  <p>Kealan Exley, Zofia Dorota Jarczynska, Linas Tamošaitis and Vijayalakshmi Kandasamy</p>
  <p>Abstract: The CfB biofoundry capabilities includes an Inscripta Onyx platform to generate massively parallel genome edited strains to accelerate the DBTL cycle. The Inscripta's automated bench-top appliance for genome engineering facilitates rapid large-scale CRISPR editing of S. cerevisiae or E. coli to introduce up to 10,000 edits. With the Inscripta Onyx platform's ability to generate thousands of strain variants within days, it is advantageous to match this pace of library generation to the identification of useful strain variants. By combining the capabilities of the Inscripta Onyx and the high-throughput platforms at the CfB Biofoundry, for targeting and identifying novel genes respectively, an optimized yeast strain producing cis,cis-muconic acid (ccM) with increased yields was generated within one month. Subsequent in-house sequencing of barcoded editing plasmid DNA from Inscripta tracked the abundance of each edit in the Inscripta library and revealed a number of unique gene targets that improved ccM yields.</p>

  <p>11:30-11:45 <strong>The iBioFoundry: Automated, Low-Cost, High-Throughput Experimentation</strong></p>
  <p>Camillo Moschner, Charlie Wedd, Georgeos Hardo and Somenath Bakshi</p>
  <p>Abstract: iBioFoundry is an open-source, low-cost automation pipeline written in Python and designed for maximal flexibility in DNA assem- bly and experiment preparations. It can be applied to a variety of different liquid handling systems and is powered by the Opentrons OT-2 robot. To enable easy access to this lab automation pipeline we showcase its use on the low cost Opentron robot. The iBioFoundry follows the implementation steps of a given application, and in version 1.0 focuses on method-agnostic DNA assembly. Further Jupyter Notebooks calculate and execute liquid handling for chemical transformations, cell spreading on multi-well plates, colony PCRs on day 2, and automated liquid culture inoculation with PCR-verified colonies in 96-deep well plates. This DNA assembly automation workflow enables low- cost creation of up to 192 plasmids by one person in a period of 3 days, starting with purified DNA and finishing with glycerol stocks of PCR-verified assemblies, and sequencing- ready samples.The iBioFoundry represents an automation pipeline, designed to democratize lab automa- tion, both in terms of financial investment and usability. The usage of Jupyter Notebooks allows for human-in-the-loop (HITL) design and liquid handling, enabling real-time feedback to the operator. The iBioFoundry counters this trend using a throughput-adaptive workflow, implemented on the low-cost Opentrons OT-2 robot.</p>

  <p>11:45 - 12:00 Short Break</p>

  <p>12:00 - 13:00 <strong>Talks Session 2: Modeling</strong> Chair: Alejandro Vignoni</p>

  <p>12:00 - 12:15 <strong>Model-driven analysis and debugging of synthetic logic circuits with new CRISPRi components</strong></p>

  <p>Davide De Marchi, Roman Shaposhnikov, Paolo Magni and Lorenzo Pasotti</p>

  <p>Abstract: Toolkits of biological parts and computer-aided methodologies to support the design and debugging of synthetic circuits are urgently needed to exploit the full potential of synthetic biology. In this work, we used the Staphylococcus aureus dead-Cas9 (SadCas9) to design synthetic circuits in engineered bacteria. This regulator was recently used in other organisms motivated by advantages over the traditionally adopted Streptococcus pyogenes dCas9, but its application to bacteria is still missing. We show that SadCas9 is suitable for synthetic circuit design such as NOT gates. However, quantitatively unexpected transfer function shapes were observed and mathematical modeling was adopted to drive the analysis and debugging of this circuitry. The lessons learned from the model system shown here demonstrate the successful use of a new biological component for bacterial synthetic biology and will be useful to biological engineers to identify sources of circuit unpredictability.<p>

  <p>12:15 - 12:30 <strong>From Specification to Implementation: Assume-Guarantee Contracts for Synthetic Biology</strong></p>

  <p>Ayush Pandey, Inigo Incer, Alberto Sangiovanni Vincentelli and Richard M Murray</p>

  <p>Abstract: Mathematical modeling has played a key role in the foundations of synthetic biology and since then has been extensively used to study the design of engineered biological systems. From a design standpoint, the development of models is necessary to decide when to use the circuits described by these models. As system complexity increases, we believe that it is necessary to develop a complete design methodology that begins with a top-level description of the system's objective and guides the designer in the generation of an implementation that can be proven to meet the specification. This is the main contribution of this paper. We present a methodology that decouples reasoning about component specifications from reasoning about the modeling details of each component. This methodology allows designers to focus on particular aspects of the design process at various levels of detail while ensuring that other aspects of the design are not forgotten.</p>

  <p>12:30 - 12:45 <strong>A Bounded Model Checking Framework for the Analysis of Chemical Reaction Network Models</strong></p>

  <p>Mohammad Ahmadi, Lukas Buecherl, Zhen Zhang, Chris Myers, Chris Winstead and Hao Zheng</p>

  <p>Abstract: Here, a new approach based on Bounded Model Checking (BMC) is proposed to analyze infinite-state CRN models.This approach is able to generate a lowerbound for the probability of certain events happening on a CRN model with potentially infinite state-space. An important byproduct of this framework is the generated witness set.We argue that reporting the mere probability of an event, although important, is not helpful enough for debugging a model. The generated witness set tends to be small (compared to the potentially infinite state-space of the model) and thus can be analyzed to extract information such as the relative frequency of reactions in error-traces. This information can be utilized to gain insight into the root cause of the erroneous behavior of the model.</p>

  <p>12:45 - 13:00 <strong>Characterization of integrase and excisionase activity in cell-free protein expression system using a modeling and analysis pipeline</strong></p>

  <p>Ayush Pandey, Makena L Rodriguez, William Poole and Richard M Murray</p>

  <p>Abstract: We present an automated and iterative pipeline for modeling, analysis, and parameter identification of biological circuits by building on existing Python tools --- BioCRNpyler, AutoReduce, and Bioscrape. Using this pipeline, we characterize the expression and activity of enzyme-mediated DNA recombination in cell-free protein synthesis system. We build detailed chemical reaction network models from high-level description of the biological circuit and the context using BioCRNpyler. We show that many sensitive parameters in this detailed model affect the output. However, for feasible parameter identification, we use AutoReduce to automatically obtain reduced models that have fewer parameters. We derive a hierarchy of reduced models under different assumptions to finally derive a minimal ODE model for each circuit. Then, we run sensitivity analysis-guided Bayesian inference using Bioscrape for each circuit to identify the parameters of the models. We characterize the strength of integrase to flip a promoter direction as well as the excisionase mechanisms to reverse it. This characterization of the integrase-excisionase activity in cell-free opens up a new paradigm of complex circuit designs that depend on accurate control over protein expression levels independent of induction or degradation.<p>

  <p>14:30 - 15:30 <strong>Talks Session 3: Measurement</strong> Chair: Zoé Pincemaille</p>

  <p>14:30 - 14:45 <strong>FPCountR: improved analytical methods enable absolute protein quantification</strong></p>
  <p>Eszter Csibra and Guy-Bart Stan</p>
  <p>Abstract: Our aim for this work is to develop a generalisable method for fluorescent protein (FP) calibration, that could be used by any group wishing to calibrate fluorescence readings on microplate readers, from arbitrary or relative units to molecular units. While methods for conducting calibrations with small molecule fluorophores are available, for this work we were interested in using fluorescent proteins directly as calibrants. We reasoned it should be possible to develop a simplified protocol for fluorescent protein purification, that allows users to directly calibrate their instruments using the same FPs as present in their cellular assays, in order to ensure that the values obtained from calibrated experimental measurements directly reflect the number of protein molecules present per cell.</p>

  <p>14:45 - 15:00  <strong>magmiX- An automated magnetic bio-separator for sustainable biomedical research</strong></p>

  <p>Christoph Sadee, Julian Alexander Zagalak, George Konstantinou and Jernej Ule</p>

  <p>Abstract: Magnetic bio-separation is widely used to isolate and enrich biomolecules or cells from complex input samples. Current magnetic bio-separation workflows suffer from a lack of efficiency, user-friendliness and reproducibility. In order to address this caveat, we have developed a multichannel-ready, automated, magnetic module (magmiX) capable of assisting researchers in their magnetic bio-separation workflows. MagmiX significantly reduces protocol tedium and increases reproducibility, while reducing protocol-associated single plastic consumption thereby making an important contribution to laboratory sustainability.</p>

  <p>15:00 - 15:15  <strong>Towards an automated assay for the quantification of secreted proteins</strong></p>
  <p>Sara Napolitano, Sebastián Sosa Carrillo, François Bertaux, Hélène Philippe and Gregory Batt</p>
  <p>Abstract: Bioproduction is a field that is increasingly gaining ground. It aims to produce goods of interest, including proteins, in biological systems, such as yeasts, which have the advantage of secreting proteins by simplifying downstream processes. Here we present a novel and general pipeline for quantifying secreted proteins, based on the use of magnetic microspheres, which allow a simplified method for separating secreted proteins from cells, and a flow cytometer, which quantifies secretion levels.</p>

  <p>15:15 - 15:30 <strong>Rapid gene circuits prototyping with JUMP assembly</strong></p>

  <p>Rizki Mardian, Marcos Valenzuela-Ortega, Jin Wong and Christopher French</p>

  <p>Abstract: A synthetic biology project requires the capability to build an extensive library of DNA constructs in order to explore the genetic design space. Modular Cloning (MoClo) can facilitate rapid assembly of hundreds of genetic constructs in a standardized and automated fashion. Here, we introduce a MoClo-based vector design named JUMP (Joint Universal Modular Plasmids) that aims to improve the existing MoClo standards, by combining the compatibility of commonly adopted vector standards, such as PhytoBricks, BioBricks, and Standard European Vector Architecture standard (SEVA). JUMP is a flexible framework because it is easy to add new features at any assembly stages via two secondary sites flanking the main insertion chassis. Additionally, we have also developed an open-source software package that can be used to design and create an assembly plan with the simple click of a button, further assisting the quick assembly of a large number of genetic constructs using JUMP. We demonstrate this capability with a case study of building combinatorial assembly of a synthetic gene circuit.</p>

  <p>15:30 - 15:45 Short Break</p>
  <p>15:45 - 16:45 <strong>Panel: Young Biofoundries</strong></p>

  <p>panelists: Daniel Schindler (MPI Marburg, MaxGENESYS), François Bertaux (Lesaffre), <del>Stéphane Lemaire (Sorbonne Université)</del> Jean-Loup Faulon (Institut Micalis/INRAE), Joan Hérisson (Université Evry — Paris-Saclay (Genoscope)), panel host: Alexis Casas (Imperial College London)</p>

  <p>16:45 - 17:00 Short Break</p>

  <p>17:00 - 18:00 <strong>Talks Session 4: Softwares and Pipelines</strong>Chair: Ayush Pandey<p>

  <p>17:00 - 17:15 <strong>GUARDIAN: Ensemble Detection of Engineering Signatures</strong>
  <p>Aaron Adler, Joel Bader, Brian Basnight, Jitong Cai, Elizabeth Cho, Joseph Collins, Yuchen Ge, John Grothendieck, Kevin Keating, Tyler Marshall, Anton Persikov, Helen Scott, Roy Siegelmann, Mona Singh, Allison Taggart, Benjamin Toll, Daniel Wyschogrod, Fusun Yaman, Eric Young and Nicholas Roehner</p>
  <p>Abstract: As part of the Guard for Uncovering Accidental Release, Detecting Intentional Alterations, and Nefariousness (GUARDIAN) project, we have developed and integrated tools that use a variety of artificial intelligence (AI) and machine learning (ML) techniques to screen sequence data and individual cells for signatures of engineering. Our whole-genome sequencing analysis system uses an ensemble approach based on the guiding principle that no single approach is likely to be able to detect engineering with perfect accuracy. Critically, ensembling enables GUARDIAN to detect foreign sequence inserts in 13 target organisms with a high degree of specificity that requires no subject matter expert (SME) review.</p>

  <p>17:15 - 17:30 <strong>SIMPLIFE: An automated pipeline for inserting functional domains into globular proteins</strong></p>
  <p>Georgie Hau Sorensen, Fabio Parmeggiani and Thomas Gorochowski</p>
Anstract: SIMPLIFE was created to simplify the design process of inserting functional domains into a globular protein (backbone), while retaining function of both insert and backbone. Designing such an insertion would usually require some parameter optimisation specific for each input structure, as well as some prior knowledge on what parts of the backbone can be altered. With SIMPLIFE, prior knowledge is not needed, since the workflow involves a step to rapidly graft and score the insert domain into every single loop of of the backbone structure. SIMPLIFE therefore functions as agnostic screening tool that returns only loops for which the insertion of an insert domain is energetically favourable.

<p>17:30 - 17:45 <strong>Galaxy-SynBioCAD: Automated Pipeline for Industrial Biotechnology</strong></p>

<p>Joan Hérisson, Thomas Duigou, Kenza Bazi-Kabbaj, Mahnaz Sabeti Azad, Manish Kushwaha and Jean-Loup Faulon</p>

<p>Abstract: We introduce the Galaxy-SynBioCAD portal, the first toolshed for synthetic biology, metabolic engineering, and industrial biotechnology. The tools and workflows currently shared on the portal enables one to build libraries of strains producing desired chemical targets covering an end-to-end metabolic pathway design and engineering process from the selection of strains and targets, the design of DNA parts to be assembled, to the generation of scripts driving liquid handlers for plasmid assembly and strain transformations. Standard formats like SBML and SBOL are used throughout to enforce the compatibility of the tools. In a study carried out at four different sites, we illustrate the link between pathway design and engineering with the building of a library of E. coli lycopene-producing strains. We also benchmarked our workflows on literature and expert validated pathways. Overall, we find an 83% success rate in retrieving the validated pathways among the top 10 pathways generated by the workflows.</p>

<p>17:45 - 18:00 <strong>Implementing Cross-Platform Protocol Execution with the Laboratory Open Protocol language</strong><p>
<p>Bryan Bartley, Jacob Beal, Daniel Bryce, Alexis Casas, Jeremy Cahill, Timothy Fallon, Robert Goldman, Luiza Hesketh, Tim Dobbs and Alejandro Vignoni</p>
<p>Abstract: Laboratory protocols are used for a wide range of purposes in research and development, at many different stages, including experiment design, execution, data analysis, interpretation, and communication and sharing with other groups. However, protocols are often difficult to communicate or reproduce, given the differences in context, skills, instruments, and other resources between different projects, investigators, and organizations.
To this end, the Bioprotocols Working Group (https://github.com/Bioprotocols) has developed a draft specification for a unified protocol modeling language, called the Laboratory Open Protocol (LabOP) language. Here we describe recent progress implementing LabOP language and demonstrating that it can be translated to and executed across different laboratory platforms in order to address use cases presented by the stakeholder community.</p>

<p>18:00 - 18:15 <strong>Closing Remarks</strong></p>

</body></html>
