<!DOCTYPE html>
<html>
  <head>
    <title>IWBDA: International Workshop on Biodesign Automation</title>
    <link href="http://fonts.googleapis.com/css?family=Exo|Open+Sans:400,600" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/bootstrap.min.css" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/main.css" media="screen" rel="stylesheet" type="text/css" />
    <script src="../javascripts/jquery-1.9.0.min.js" type="text/javascript"></script>
    <script src="../javascripts/bootstrap.min.js" type="text/javascript"></script>
  </head>
  <body>
    <div id="strip"></div>
    <div class="container">
      <div class="row" id="logo">
        <div class="span8">
          <a class="brand" href="/2016/"><h3>IWBDA 2016</h3></a>
        </div>
        <div class="span4">
          <h4 class="pull-right">August 16-18, 2016</h4>
        </div>
      </div>
      <div class="row" id="nav">
        <div class="span12">
          <ul class="nav nav-pills">
            <li><a href="/2016/#content">About</a></li>
            <li><a href="/2016/#sponsors">Sponsors</a></li>
            <li><a href="/2016/#reg">Registration</a></li>
	    <li><a href="/2016/#cfp">Call for Papers</a></li>
            <li><a href="/2016/#hotel">Accommodations</a></li>
            <li><a href="/2016/sponsorship/">Sponsorship</a></li>
            <li><a href="/2016/#bdathlon">BDAthlon</a></li>
            <li><a href="/2016/#keynotes">Keynote Speakers</a></li>
            <li class="active"><a href="/2016/program/">Program</a></li>
            <li><a href="/2016/#dinner">Annual IWBDA Dinner</a></li>
            <li><a href="/2016/#scholarships">Scholarships</a></li>
            <li><a href="/2016/#special">Special Issue</a></li>
            <li><a href="/2016/#organizing">Contact</a></li>
            <li><a href="/2016/#related">Related Conferences</a></li>
            <li><a href="/archive/">Archive</a></li>
          </ul>
        </div>
    </div>
    <div class="row">
      <div class="jumbotron">
	<div class="span12">
          <img src="../images/logo.png" alt="IWBDA 2016: 8th International Workshop on Bio-Design Automation, Newcastle University, Newcastle upon Tyne, UK"/>
        </div>
	<div class="span12">
	  <h3 class="pull-right">8th International Workshop on Bio-Design Automation</h3>
	</div>
      </div>
    </div>

      <br>

      <div id="content">

	<h4 id="proceedings">Proceedings</h4>
	<p><a href="/2016/docs/IWBDA_2016_Proceedings.pdf">Link to proceedings pdf.</a></p>
	<p><a href="/2016/docs/IWBDA_2016_Supplemental.pdf">Link to proceedings supplemental pdf.</a></p>
	<hr> 

	<h4 id="toc_0">Program</h4>

	<p>
	  The following is the preliminary schedule, subject to
	  change.<br>  
	  The schedule of abstracts to be presented
	  as <a href="#talks">talks</a>
	  and <a href="#posters">posters</a>. Detailed <a href="#instructions">instructions
	  for presenters</a> are below.
	</p>

	<hr>
	<h4 id="talks">Agenda</h4>(Updated on August 7, 2016)
	
	<p>
	  <table cellpadding=0 cellspacing=0 border=0 width=100%>

	    <tr>
              <th class=session colspan=2 align=left>Monday, August 15th</th>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>09:00 - 18:30</td>
	      <td>15th SBOL Workshop<br>
Prior to IWBDA, there will be a one-day SBOL workshop on August 15th,
2016 at Newcastle University, Newcastle upon Tyne, UK. The workshop is
aimed for software developers, and will include demos and
tutorials. Although there is no registration fee for the SBOL
workshop, participants should still register.<br>
For details about this workshop and the registration please click here: <a href="http://sbolstandard.org/1226-2/">http://sbolstandard.org/1226-2/</a>.
	      </td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th class=session colspan=2 align=left>Tuesday, August 16th</th>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>08:00 - 00:00 (August 17th)</td>
	      <td>1st BDAthlon programming contest</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th class=session colspan=2 align=left>Wednesday, August 17th</th>
	    </tr>
	    <tr>
	      <td width=12% valign=top align=center>8:00 - 8:30</td>
	      <td>Arrival, Breakfast, and Registration</td>
	    </tr>
	    <tr>
              <td width=12% valign=top align=center>8:30 - 8:40</td>
	      <td>Opening Remarks<br/>
		Anil Wipat and Pietro Lio', Co-General Chairs</td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr>
              <th class=session colspan=2>Talk Session I: Logic, Moderator: Nic Roehner</th>
	    </tr>
	    <tr>
              <td valign=top align=center>8:40 - 9:00</td>
	      <td>Design for Improved Repression in RNA Replicons<br/>
		Jacob Beal and Ron Weiss
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:00 - 9:20</td>
	      <td>Noise Tolerance Analysis for Reliable Analog Computation in Living Cells<br/>
		Ramez Danial
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:20 - 9:40</td>
	      <td>Utilizing Signal Temporal Logic to Characterize and Compose Modules in Synthetic Biology<br/>
		Curtis Madsen, Prashant Vaidyanathan, Cristian-Ioan Vasile, Rachael Ivison, Junmin Wang, Calin Belta, and Douglas Densmore
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:40 - 10:00</td>
	      <td>Single Cell Analysis of RNA-engineered Logic Gates<br/>
		Christopher Schneider, Jascha Diemer, Leo Bronstein, Heinz Koeppl, and Beatrix Suess
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>10:00 - 10:30</td>
	      <td>Coffee Break</td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr><th class=session colspan=2>Keynote I</th></tr>
	    <tr>
              <td valign=top align=center>10:30 - 11:30</td>
	      <td><span class="speaker">Dr. Amoolya Singh</span>
	      Automating Design at an Industrial Biotech<br> 
		Abstract:
	      Amyris has developed a high-throughput genetic
	      engineering platform for designing and building custom
	      microbes to serve as living factories. Using an
	      industrial scale fermentation process, our microbes
	      convert cheap sugars into a wide variety of high value
	      target molecules, including medicines, commodity and
	      specialty chemicals. Our end products provide low cost,
	      high quality malaria medication and renewable
	      substitutes for fuels and chemicals. Amyris' R&D efforts
	      span rational & random design and construction of
	      microbial strains, high-throughput screening and
	      analytical chemistry, fermentation at multiple scales,
	      and genotype/phenotype data mining. Every aspect of this
	      work is facilitated and accelerated by quantitative
	      science and software & hardware automation.  In this
	      talk, I will outline the computational challenges
	      inherent in automating the design of microbial strains.
	      To meet these challenges, we use a range of innovations
	      including genotype specification languages and
	      high-level functional ontologies of parts and pathways;
	      metabolic and statistical models; literature mining
	      algorithms; and design of experiments approaches.</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Talk Session II: Tools I, Moderator: Traci Haddock</th></tr>
	    <tr>
              <td valign=top align=center>11:30 - 11:50</td>
	      <td>BioBlocks: A Web-Based Visual Environment for Programming Experimental Protocols in Biological Sciences<br/>
		Vishal Gupta, Jes&uacute;s Irimia, Iv&aacute;n Pau, and Alfonso Rodr&iacute;guez-Pat&oacute;n
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>11:50 - 12:10</td>
	      <td>An Environment for Augmented Biodesign Using Integrated Data Resources<br/>
		James McLaughlin, Goksel Misirli, Matthew Pocock, and Anil Wipat	
	      </td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Poster Pitches</th></tr>
	    <tr>
              <td valign=top align=center>12:10 - 12:30</td>
	      <td>Poster Pitches: 1 minute per poster</td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Lunch</th></tr>
	    <tr>
              <td valign=top align=center>12:30 - 12:40</td>
	      <td>Announcements</td>
	    </tr>
	    <tr>
              <td valign=top align=center>12:40 - 13:10</td>
	      <td>Lunch</td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr><th class=session colspan=2>Poster Session and Demos</th></tr>
	    <tr>
              <td valign=top align=center>13:10 - 14:10</td>
	      <td>Poster Session</td>
	    </tr>
            
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr>
              <th class=session colspan=2>Talk Session III: Pathways, Moderator: Evan Appleton</th>
	    </tr>
	    <tr>
              <td valign=top align=center>14:10 - 14:30</td>
	      <td>extFogLight: Using Weighted Metabolic AND/OR Graph to â€ŽFind Stoichiometric Balanced Pathways<br/>
		Mehrshad Khosraviani and Morteza Saheb Zamani
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>14:30 - 14:50</td>
	      <td> PathwayGenie - Pathway Design from Selection to Plasmid<br/>
		Neil Swainston, Pablo Carbonell, Adrian Jervis, Christopher Robinson, Mark Dunstan, and Jean-Loup Faulon
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>14:50 - 15:10</td>
	      <td> Integrated Predictive Genome-Scale Models to Improve the Metabolic Re-Engineering Efficiency<br/>
		Vishwesh Kulkarni, Lina El Menjra, Pablo Carbonell, and Jean-Loup Faulon
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>15:10 - 15:30</td>
	      <td>SBCDOE: a Design of Experiments-based Part Planner for Synthetic Biology Production of Chemicals<br/>
		Mark Dunstan, Adrian Jervis, Christopher Robinson, Neil Swainston, Jean-Loup Faulon, and Pablo Carbonell
	      </td>
	    </tr>
	    
	    <tr>
              <td valign=top align=center>15:30 - 15:40</td>
	      <td>Coffee Break</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Evening Activities</th></tr>
	    <tr>
              <td valign=top align=center>15:40 - 15:45</td>
	      <td>Coaches leave at Centre for Life</td>
	    </tr>
	    <tr>
              <td valign=top align=center>17:00 - 18:00</td>
	      <td>Durham Castle tour</td>
	    </tr>
	    <tr>
              <td valign=top align=center>18:00 - 19:00</td>
	      <td>Durham Cathedral tour</td>
	    </tr>
	    <tr>
              <td valign=top align=center>19:30 - 22:30</td>
	      <td>Conference Dinner at the Durham Castle. Coaches depart at 10:30 PM.</td>
	    </tr>
            
	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th colspan=2 align=left>Thursday, August 18th</th>
	    </tr>

	    <tr>
	      <td width=12% valign=top align=center>8:00 - 8:30</td><td>Arrival, Breakfast, and Registration</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th class=session colspan=2>Talk Session IV: Tools II, Moderator: Jacob Beal</th>
	    </tr>
	    <tr>
              <td valign=top align=center>8:30 - 8:50</td>
	      <td>A Web-Based Validator and Validation API for the Synthetic Biology Open Language<br/>
		Zach Zundel, Meher Samineni, Zhen Zhang, and Chris Myers
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>8:50 - 9:10</td>
	      <td>SBOLDesigner 2.0<br/>
		Michael Zhang and Chris Myers
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:10 - 9:30</td>
	      <td>Bioform: an in-silico 3D Physical Modelling Platform for the Design and Analysis of Bacterial Populations<br/>
		Jonathan Naylor, Harold Fellermann, Waleed Mohammed, Nick Jakubovics, Joy Mukherjee, Catherine Biggs, Phillip Wright, and Natalio Krasnogor
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:30 - 9:50</td>
	      <td>TEBio - Tools for Engineering Biology<br/>
		James Scott-Brown, Thomas Prescott, and Antonis Papachristodoulou
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>9:50 - 10:20</td>
	      <td>Coffee Break</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Keynote II</th></tr>
	    <tr>
              <td valign=top align=center>10:20 - 11:20</td>
	      <td><span class="speaker">Prof. Natalio Krasnogor</span>
		Accelerating Synthetic Biology via Software and Hardware Advances<br>  
		Abstract: In this talk I will discuss
		recent work done in my lab that contributes towards
		accelerating the specify -&gt; design -&gt; model -&gt; build
		-&gt; test & iterate biological engineering cycle. This
		will describe advances in biological programming
		languages for specifying combinatorial DNA libraries,
		the utilisation of off-the-shelf microfluidic devices
		to build the DNA libraries as well as data analysis
		techniques to accelerate computational simulations.
              </td>

	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>


	    <tr>
              <th class=session colspan=2>Talk Session V: Standards, Moderator: Harold Fellermann</th>
	    </tr>
	    <tr>
              <td valign=top align=center>11:20 - 11:40</td>
	      <td>ShortBOL: A shorthand for SBOL<br/>
		Matthew Pocock, Chris Taylor, Goksel Misirli, James McLaughlin, and Anil Wipat
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>11:40 - 12:00</td>
	      <td>A Data Model for the Description of Bioparts<br/>
		I&ntilde;aki Sainz de Murieta, Matthieu Bultelle, and Richard I. Kitney	
	      </td>
	    </tr>
	    
	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr><th class=session colspan=2>Lunch</th></tr>
	    <tr>
              <td valign=top align=center>12:00 - 13:30</td>
	      <td>Lunch</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr>
              <th class=session colspan=2>Talk Session VI: Automation, Moderator: Luis Ortiz</th>
	    </tr>
	    <tr>
              <td valign=top align=center>13:30 - 13:50</td>
	      <td>Design and Automated Inference of Design Principles in Gene Regulatory Networks: a Multiobjective Optimization Approach<br/>
		Irene Otero-Muras and Julio R. Banga
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>13:50 - 14:10</td>
	      <td>How to Remember and Revisit Many Genetic Design Variants Automatically<br/>
		Nicholas Roehner and Douglas Densmore
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>14:10 - 14:30</td>
	      <td>Towards Automated Biosecurity: Screening of Synthetic Biology Constructs<br/>
		Benjamin Apra, Arthur Vigil, and James Diggans
	      </td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Discussion Session, Leader: Doug Densmore</th></tr>
	    <tr>
              <td valign=top align=center>14:30 - 15:30</td>
	      <td>Discussion Topic: BDA Hackathons and Community Engagementâ€‹</td>
	    </tr>

	    <tr>
              <td valign=top align=center>15:30 - 16:00</td>
	      <td>Coffee Break</td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>
	    
	    <tr>
              <th class=session colspan=2>Talk Session VII: Circuits, Moderator: Goksel Misirli</th>
	    </tr>
	    <tr>
              <td valign=top align=center>16:00 - 16:20</td>
	      <td>Realization of Large Logic Circuits with Long-Term Memory Using CRISPR/Cas9 Systems<br/>
		Tai-Yin Chiu, Cheng-Han Hsieh and Jie-Hong Roland Jiang
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>16:20 - 16:40</td>
	      <td>3D Printing of Microbes for Material Production<br/>
		Benjamin Lehner and Anne S. Meyer
	      </td>
	    </tr>
	    <tr>
              <td valign=top align=center>16:40 - 17:00</td>
	      <td>MakerFluidics: Microfluidics for the Masses<br/>
		Ryan Silva, Radhakrishna Sanka, and Douglas Densmore
	      </td>
	    </tr>

	    <tr><td>&nbsp;</td><td></td></tr>

	    <tr><th class=session colspan=2>Closing Remarks and Awards</th></tr>
	    <tr>
              <td valign=top align=center>17:00 - 17:15</td>
	      <td>Closing remarks<br/>
		Anil Wipat and Pietro Lio', Co-General Chairs</td>
	    </tr>
	  </table>

	  <hr/>

	<h4 id="posters">Posters</h4>
	<p>
          <table cellpadding=10 cellspacing=0 border=0 width=100%>
            <tr><td>Bacterial Detoxification of Martian Soil Coupled to Oxygen Production<br/>Valentijn Broeken and Lisanne van Oosterhoud</td></tr>
            <tr><td>Business Process Management of Synthetic Biology Workflows<br/>Christopher Reynolds, Jocelyne Holdbrook, Kealan Exley, and Richard Kitney</td></tr>
            <tr><td>CVLTool: Oligonucleotide Design Software for Assembly of Orthogonal Codon Variant Gene Libraries<br/>Surya Teja Chinta and Dimitris Papamichail</td></tr>
            <tr><td>Genetic Systems Engineering<br/>Prashant Vaidyanathan, Evan Appleton, Curtis Madsen, Cristian-Ioan Vasile, Alan Pacheco, Iman Haghighi, Nicholas Roehner, Rachael Ivison, Junmin Wang, Yash Agarwal, Zachary Chapasko, Calin Belta, and Douglas Densmore</td></tr>
            <tr><td>Improving Controllability of Biosynthesis of Gold Nanoparticles by Shewanella oneidensis through Genetic Manipulation<br/>Juliano Bertozzi Silva, Gregory Fowler, and Phillip Wright</td></tr>
            <tr><td>Load Capacity Improvements in Transcriptional Systems Using Discrete-Time L1-Adaptive Control<br/>Hamidreza Jafarnejadsani, Jongmin Kim, Naira Hovakimyan, and Vishwesh Kulkarni</td></tr>
            <tr><td>Logic and Timing Analysis of Genetic Logic Circuits using D-VASim<br/>Hasan Baig and Jan Madsen</td></tr>
            <tr><td>MINT - A MIcrofluidic NetlisT Format<br/>Radhakrishna Sanka, Haiyao Huang, Ryan Silva, and Douglas Densmore</td></tr>
            <tr><td>Modular Assembly of an Electronically Integrated Genetic Circuit Library<br/>Luis Ortiz, Thomas Costa, and Douglas Densmore</td></tr>
            <tr><td>Modular Composition of Synthetic Biology Designs using Rule-Based Models<br/>Goksel Misirli, William Waites, Matteo Cavaliere, Paolo Zuliani, Vincent Danos, Anil Wipat, and Ricardo Honorato-Zimmer</td></tr>
            <tr><td>Owl v2.0: A Web-Application Workspace for Synthetic Biology<br/>Yury Ivanov, Prashant Vaidyanathan, Evan Appleton, Zach Chapasko, Arash Khospavar, and Douglas Densmore</td></tr>
            <tr><td>Phagebook Alpha: A Software Environment for Social Synthetic Biology<br/>Johan Ospina, Inna Turshudzhyan, Allison Durkan, Kristel Tan, Anna Goncharova, Prashant Vaidyanathan, Nicholas Roehner, and Douglas Densmore</td></tr>
            <tr><td>A Powerful, Open Source, Extensible, Cloud Tool to Drive Biological Design and Construction<br/>Edinburgh Genome Foundry, Autodesk Bionano Research, and Eli Groban</td></tr>
            <tr><td>A Statistical Approach Reveals Designs for the Most Robust Stochastic Gene Oscillators<br/>Mae Woods, Miriam Leon, Ruben Perez-Carrasco, and Chris Barnes</td></tr>
            <tr><td>SynBIS â€“ The Synthetic Biology Information System<br/>Matthieu Bultelle, I&ntilde;aki Sainz de Murieta, and Richard Kitney</td></tr>
          </table>

	  <hr/>

	  <h4 id="sponsor_posters">Sponsor Posters</h4>
	  
	<p/>

        <table cellpadding=10 cellspacing=0 border=0 width=100%>                                                                            <tr><td>Twist Bioscience</td></tr>
	</table>

        <hr/>



	  <h4 id="instructions">Instructions for Oral and Poster Presentations</h4>
	  <h5>Oral presentations</h5>
	<p>Oral presentations will each be 15 minutes + 5 minutes of Q&A. 
	  <!-- We will be providing a projector and a laptop computer
	       for the presentations. Please bring your slides on a USB stick
	       at least 10 minutes before your session begins so that we can
	       load them on our laptop computer to minimize delay between
	       presentations. -->
	</p>
	
	<h5>Poster presentations</h5>
	<p><b>Posters should be in a horizontal orientation, 40 inches wide by 30 inches tall.</b></p>
	<hr/>
	
      </div>
    </div>
  </body>
</html>
