<!DOCTYPE html>
<html>
  <head>
    <title>IWBDA: International Workshop on Biodesign Automation</title>
    <meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
    <link href="http://fonts.googleapis.com/css?family=Exo|Open+Sans:400,600" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/bootstrap.min.css" media="screen" rel="stylesheet" type="text/css" />
    <link href="../stylesheets/main.css" media="screen" rel="stylesheet" type="text/css" />
<!--    <script src="javascripts/jquery-3.1.0.min.js" type="text/javascript"></script>
    <script src="javascripts/bootstrap.min.js" type="text/javascript"></script> -->
  </head>
  <body>
    <div id="strip"></div>
    <div class="container">
      <div class="row" id="logo">
        <div class="span8">
          <a class="brand" href="/2021/"><h3>IWBDA 2021</h3></a>
        </div>
        <div class="span4">
          <h4 class="pull-right">September 20 - September 24, 2021</h4>
        </div>
      </div>
      <div class="row" id="nav">
        <div class="span12">
          <ul class="nav nav-pills">
            <li class="active"><a href="/2021/#content">About</a></li>
	          <li><a href="/2021/#reg">Registration</a></li>
            <!--<li><a href="#program">Program</a></li>-->
            <li><a href="/2021/#cfp">Call for Papers</a></li>
            <li><a href="/2021/#cfw">Call for Workshops</a></li>
            <li><a href="/2021/program">Program</a></li>
            <li><a href="/2021/workshops">Workshops</a></li>
            <li><a href="/2021/acssynbio">Special Issue</a></li>
            <li><a href="/2021/#twitter">#IWBDA2021</a></li>
            <li><a href="/2021/#organizing">Contact</a></li>
            <li><a href="/2021/#archive">Archive</a></li>
          </ul>
        </div>
    </div>
    <div class="row">
      <div class="jumbotron">
	      <div class="span12">
	        <img src="../images/IWBDA2021.png" alt="IWBDA 2021: 13th International Workshop on Bio-Design Automation, Online"/>
        </div>
	      <div class="span12">
	        <h3 class="pull-right">13th International Workshop on Bio-Design Automation</h3>
        </div>
      </div>
    </div>

    <div class="span12">
      <hr id="calendar">
      <h3>IWBDA 2021 Calendar</h3>
      <p>Please click on any event to see additional details or to add any session to your calendar.</p>
      <iframe src="https://calendar.google.com/calendar/embed?height=600&wkst=2&bgcolor=%23ffffff&ctz=Europe%2FLondon&src=dWQ4bjhnajRpMHAwbDE3bWt1OGdoajBhMGdAZ3JvdXAuY2FsZW5kYXIuZ29vZ2xlLmNvbQ&color=%23D50000&showNav=0&mode=AGENDA&showCalendars=0" style="border:solid 1px #777" width="800" height="600" frameborder="0" scrolling="no"></iframe>
      

      <hr id="detailedprogram">
      <h3>IWBDA 2021 Detailed Program</h3>
      <p>All times BST. Conference to be conducted over Zoom.</p>

  <p><strong><u>Monday, 20th September 2021</u></strong></p>

  <p>15:00 - 15:15 <strong>Welcome & Opening Remarks</strong> Prashant Vaidyanathan</p>
  
  <p>15:15 - 16:45 <strong>Session 1: Standardization of Biological Components</strong>, Chair: Andrea Cristina</p>
  <ul>
    <li>15:15-15:30 <i>Network visualisation of synthetic biology designs</i><br/>
      Matthew Crowther, Anil Wipat and Ángel Goñi-Moreno</li>
    <li>15:30-15:55 <i>Data Representation in the DARPA SD2 Program</i><br/>
      Nicholas Roehner, Jacob Beal, Bryan Bartley, Richard Markeloff, Tom Mitchell, Tramy Nguyen, Daniel Sumorok, Nicholas Walczak, Chris Myers, Zach Zundel, James Scholz, Benjamin Hatch, Mark Weston and John Colonna-Romano</li>
    <li>15:55-16:20 <i>Excel-SBOL Converter: Creating SBOL from Excel Templates and Vice Versa</i><br/>
      Julian Abam, Jeanet Mante, Isabel Pötzsch, Jake Beal and Chris Myers</li>
    <li>16:20-16:45 <i>Towards collaborative and automated development of resources for data standards in synthetic biology</i><br/>
      Jake Sumner Ajibode, Jacob Beal, James Scott-Brown, Thomas Gorochowski, Chris Myers and Goksel Misirli</li>
   </ul>

  <p>16:45 - 17:00 <strong>Short Break</strong></p>

  <p>17:00 - 18:00 <strong>Live Keynote I: Dr. Tijana Radivojevic</strong></p>
  <div style="text-align: justify">
    <ul style="list-style-type:none;">
      <li><strong>Title:</strong>&nbsp;Guiding synthetic biology via machine learning and multi-omics technologies.</li>
      <li><strong>Abstract:</strong>&nbsp;Synthetic biology allows us to bioengineer cells to synthesize novel valuable molecules such as renewable biofuels or anticancer drugs. However, traditional synthetic biology approaches involve ad-hoc engineering practices, which lead to long development times. One of the most important challenges in bioengineering is effectively using multi-omics data to guide metabolic engineering towards higher production levels. In this talk, I will show our efforts in developing pipelines for collection of multi-omics datasets, their analysis through machine learning, and the production of recommendations, with the goal of accelerating the Design-Build-Test-Learn (DBTL) cycle. I will focus on machine learning techniques that, trained on the multi-omics datasets, provide actionable recommendations predicted to optimize strain performance and increase production through several DBTL cycles. These tools help guide synthetic biology in a systematic fashion, without the need for a full mechanistic understanding of the biological system. Our tools also aim to enable commercially-relevant bioengineering and are currently being deployed by industrial and academic partners.</li>
    </ul>
  </div>
  
  <p>18:00 - 18:30 <strong><a href="https://www.gather.town/" target="_blank">Gather</a> Social Hour</strong></p>

  <p>18:30 - 20:30 <strong><a href="/2021/workshops#sbol3" target="_blank">Workshop 1</a>: SBOL Version 3: Data Exchange throughout the Bioengineering Lifecycle</strong></p>

  <br>
  <p><strong><u>Tuesday, 21st September 2021</u></strong></p>
  
  <p>15:00 - 15:05 <strong>Welcome & Opening Remarks</strong> Jenhan Tao</p>
  <p>15:05 - 16:00 <strong>Session 2: Screening Methods</strong>, Chair: Kenza Samlali</p>
  <ul>
    <li>15:05-15:30 <i>The Bioware Cyber-Fluidic Platform: A Holistic Approach to Digital Microfluidics</i><br/>
      Georgi Tanev, Luca Pezzarossa and Jan Madsen</li>
    <li>15:30-15:45 <i>An Investigative Platform Comprising Cell-Free Transcription- Translation and Electron Microscopy for Studying Bacteriophages</i><br/>
      Joseph Wheatley, Sahan Liyanagedera, Ian Hands-Portman, Antonia Sagona and Vishwesh Kulkarni</li>
    <li>15:45-16:00 <i>Engineering SpyTag Bacteriophage K1F for Directional Immobilisation</i><br/>
      Sahan Liyanagedera, Joseph Wheatley, Alyona Biketova, Ian Hands-Portman, Antonia Sagona, Kevin Purdy, Tamas Feher and Vishwesh Kulkarni</li>
    </ul>

  <p>16:00 - 16:15 <strong>Short Break</strong></p>
  <p>16:15 - 17:15 <strong>Live Keynote II: Dr. Thomas E. Gorochowski</strong></p>
    <div style="text-align: justify">
      <ul style="list-style-type:none;">
        <li><strong>Title:</strong>&nbsp;Programming biology – controlling the flow of molecular machines empowering life.</li>
        <li><strong>Abstract:</strong>&nbsp;Synthetic genetic circuits are composed of many interconnected parts that must control the flows of transcriptional and translational machinery such that a desired biological computation can be implemented. A major challenge when developing such circuits is that the genetic parts used often display unexpected changes in their behavior when pieced together in new ways. Such changes can arise due to contextual effects or unintended interactions with the host cell. In this talk, I will demonstrate how we have been using a variety of sequencing technologies to create a genetic debugger to pinpoint the root of such failures, as well as our recent efforts to develop “tunable” genetic parts whose functions can be dynamically altered to fix many of these common issues. I will also discuss some of our recent efforts to consider the role of evolution in biological design and the concept of the 'evotype' as a way to reason about the evolutionary potential of engineered biology. Taken together, our work provides a more complete and quantitative view of the inner workings of genetic circuits, offers a route to engineering more robust and adaptive functionalities in living cells, and improves our understanding of the rules governing the effective reprogramming of biology.</li>
      </ul>
    </div>
    <p>17:15 - 17:45 <strong><a href="https://www.gather.town/" target="_blank">Gather</a> Social Hour</strong></p>
    <p>17:45 - 19:45 <strong><a href="/2021/workshops#biocrnpyler" target="_blank">Workshop 2: Part I - </a> From Chemical Reaction Network Compilation to Bayesian Parameter Inference (Compilation with BioCRNpyler)</strong></p>
    <p>19:45 - 21:45 <strong><a href="/2021/workshops#biocrnpyler" target="_blank">Workshop 2: Part II - </a> From Chemical Reaction Network Compilation to Bayesian Parameter Inference (Model Reduction and Inference)</strong></p>
    
    <br>
    <p><strong><u>Wednesday, 22nd September 2021</u></strong></p>

    <p>15:00 - 15:05 <strong>Welcome & Opening Remarks</strong> Marilene Pavan</p>
    <p>15:05 - 15:55 <strong>Session 3: Metabolic & Knowledge Engineering I</strong>, Chair: Alejandro Vignoni</p>
    <ul>
      <li>15:05 - 15:30<i>Modeling of the engineered production of curcumin in Escherichia coli</i><br/>
        Michael Cotner, Ellie Brown, Jixun Zhan and Zhen Zhang</li>
      <li>15:30 - 15:55<i>Codon-Optimized Degenerate Codon Set Design Tool</i><br/>
        Akira Takada, Tomer Aberbach, Nicholas Carpino, Georgios Papamichail and Dimitris Papamichail</li>
    </ul>

    <p>15:55 - 16:00 <strong>Short Break</strong></p>

    <p>16:00 - 17:30 <strong><a href="/2021/workshops#mlsynbio" target="_blank">Workshop 3</a>: Machine Learning Aided Advances in Synthetic Biology</strong></p>

    <p>17:30 - 17:45 <strong>Short Break</strong></p>

    <p>17:45 - 18:50 <strong>Session 4: Metabolic & Knowledge Engineering II</strong>, Chair: Dimitris Papamichail</p>
    <ul>
      <li>17:45 - 18:00 <i>iBioSim Server: a Tool for Improving the Workflow for Genetic Design and Modeling</i><br/>
        Thomas Stoughton, Lukas Buecherl, Payton Thomas, Pedro Fontanarrosa and Chris Myers</li>
      <li>18:00 - 18:25 <i>Optimizing and Classifying Literature Events for Automated Model Extension</i><br/>
        Casey Hansen, Julia Kisslinger, Neal Krishna, Emilee Holtzapple, Yasmine Ahmed and Natasa Miskov-Zivanov</li>
      <li>18:25 - 18:50 <i>New advances in the automation of context-aware information selection and guided model assembly</i><br/>
        Yasmine Ahmed, Adam A. Butchy, Khaled Sayed, Cheryl Telmer and Natasa Miskov-Zivanov</li>
    </ul>
    <p>18:50 - 19:15 <strong><a href="https://www.gather.town/" target="_blank">Gather</a> Social Hour</strong></p>

    <p>19:15 - 21:15 <strong><a href="/2021/workshops#3duf" target="_blank">Workshop 4</a>: DIY Microfluidics CAD - Extending 3DuF for fun and publications</strong></p>

    <br>
    <p><strong><u>Thursday, 23rd September 2021</u></strong></p>
    <p>15:00 - 15:05 <strong>Welcome & Opening Remarks</strong> Alexis Casas</p>
    <p>15:05 - 16:45 <strong>Session 5: Computer-aided design, modelling, and simulation</strong>, Chair: Martin G. Pescarmona</p>
    <ul>
      <li>15:05 - 15:30 <i>A Comparison of Weighted Stochastic Simulation Methods</i><br/>
        Payton Thomas, Mohammad Ahmadi, Hao Zheng and Chris Myers</li>
      <li>15:30 - 15:55 <i>LOICA: Logical Operators for Integrated Cell Algorithms</i><br/>
        Gonzalo Vidal, Guillermo Yáñez-Feliú, Carlos Vidal-Céspedes and Timothy James Rudge</li>
      <li>15:55 - 16:20 <i>Biophysical Technology Mapping of Genetic Circuits</i><br/>
        Nicolai Engelmann, Tobias Schladt, Erik Kubaczka, Christian Hochberger and Heinz Koeppl</li>
      <li>16:20 - 16:45 <i>Automated translation of logical models to SystemVerilog enables simulation speedup</i><br/>
        Eric Li, Emilee Holtzapple, Niteesh Sundaram and Natasa Miskov-Zivanov</li>
    </ul>

    <p>16:45 - 17:15 <strong> Break / Short Discussion - Distributed mini-biofoundries</strong></p>

    <p>17:15 - 21:15 <strong><a href="/2021/workshops#flapjack" target="_blank">Workshop 5</a>: Flapjack: Data Management and Analysis for Genetic Circuit Characterization</strong></p>
    <br>
    <p><strong><u>Friday, 24th September 2021</u></strong></p>
    <p>14:00 - 14:05 <strong>Welcome & Opening Remarks</strong></p>
    <p>14:05 - 14:50 <strong><a href="/2021/workshops#bmss" target="_blank">Workshop 6</a>: BMSS - An Automated BioModel Selection System for Gene Circuit Designs</strong></p>
    <p>14:50 - 15:00 <strong>Short Break</strong></p>
    <p>15:00 - 15:40 <strong>Session 6: Machine Learning</strong>, Chair: Jeanet Mante</p>
    <ul>
      <li>15:00 - 15:15 <i>Improving predictability in bio-design using ensemble models</i><br/>
        Bret Peterson, Tijana Radivojevic and Hector Garcia Martin</li>
      <li>15:15 - 15:40 <i>Comparison of Extrinsic and Intrinsic Noise Model Predictions for Genetic Circuit Failures</i><br/>
        Pedro Fontanarrosa, Lukas Büecherl and Chris J. Myers</li>
    </ul>
    <p>15:40 - 16:00 <strong><a href="https://www.gather.town/" target="_blank">Gather</a> Social Hour</strong></p>
    <p>16:00 - 18:00 <strong><a href="/2021/workshops#sbolvisual" target="_blank">Workshop 7</a>: Visualizing biological designs using SBOL visual</strong></p>
    <p>18:00 - 18:15 <strong>Short Break</strong></p>
    <p>18:15 - 18:45 <strong>Session 7: Data Repositories</strong>, Chair: Samuel MD Oliveira</p>
    <ul>
      <li>18:15 - 18:30 <i>SynBioHub2 - Providing an Intuitive and Maintainable Genetic Design Repository</i><br/>
        Benjamin Hatch, Jeanet Mante, Chris Myers and Eric Yu</li>
      <li>18:30 - 18:45 <i>A database for ligand-inducible genetic biosensors</i><br/>
        Simon d'Oelsnitz</li>
    </ul>
    <p>18:45 - 20:45 <strong><a href="/2021/workshops#circuitdesign" target="_blank">Workshop 8</a>: Principles of genetic circuit design: programming living cells to perform novel functions</strong></p>
    <p>20:45 - 21:00 <strong>Closing Remarks</strong></p>

    </div>
  </div>
    
</body></html>
